{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import pandas as pd\n",
    "from tabulate import tabulate\n",
    "from great_tables import GT\n",
    "import zentables as zen\n",
    "import dataframe_image as dfi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "Data = pd.read_csv(\"../Data/topVarGenes.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/alec/miniconda3/envs/dCasRx/lib/python3.12/site-packages/IPython/core/formatters.py:347: FutureWarning: Index.format is deprecated and will be removed in a future version. Convert using index.astype(str) or index.map(formatter) instead.\n",
      "  return method()\n",
      "/Users/alec/miniconda3/envs/dCasRx/lib/python3.12/site-packages/IPython/core/formatters.py:347: FutureWarning: RangeIndex.format is deprecated and will be removed in a future version. Convert using index.astype(str) or index.map(formatter) instead.\n",
      "  return method()\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>column 1</th>\n",
       "      <th>Gene products (complexes, polypeptides, and RNAs)</th>\n",
       "      <th>GO terms (biological process)</th>\n",
       "      <th>Pathways of gene</th>\n",
       "      <th>Binding sites upstream of gene</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ymdF</td>\n",
       "      <td>stress-induced bacterial acidophilic repeat mo...</td>\n",
       "      <td>cell adhesion involved in single-species biofi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>lolC</td>\n",
       "      <td>lipoprotein release complex</td>\n",
       "      <td>protein localization to outer membrane // lipo...</td>\n",
       "      <td>lipoprotein posttranslational modification (Gr...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>rho</td>\n",
       "      <td>transcription termination factor Rho</td>\n",
       "      <td>DNA-templated transcription termination</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ssrA</td>\n",
       "      <td>tmRNA</td>\n",
       "      <td>rescue of stalled ribosome // trans-translatio...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>yibJ</td>\n",
       "      <td>putative RHS domain-containing protein YibJ</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  column 1  Gene products (complexes, polypeptides, and RNAs)   \n",
       "0     ymdF  stress-induced bacterial acidophilic repeat mo...  \\\n",
       "1     lolC                        lipoprotein release complex   \n",
       "2      rho               transcription termination factor Rho   \n",
       "3     ssrA                                              tmRNA   \n",
       "4     yibJ        putative RHS domain-containing protein YibJ   \n",
       "\n",
       "                       GO terms (biological process)   \n",
       "0  cell adhesion involved in single-species biofi...  \\\n",
       "1  protein localization to outer membrane // lipo...   \n",
       "2            DNA-templated transcription termination   \n",
       "3  rescue of stalled ribosome // trans-translatio...   \n",
       "4                                                NaN   \n",
       "\n",
       "                                    Pathways of gene   \n",
       "0                                                NaN  \\\n",
       "1  lipoprotein posttranslational modification (Gr...   \n",
       "2                                                NaN   \n",
       "3                                                NaN   \n",
       "4                                                NaN   \n",
       "\n",
       "  Binding sites upstream of gene  \n",
       "0                            NaN  \n",
       "1                            NaN  \n",
       "2                            NaN  \n",
       "3                            NaN  \n",
       "4                            NaN  "
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "topVarGenes = [\n",
    "    \"ymdF\", \"lolC\", \"rho\", \"ssrA\", \"yibJ\", \"intA\", \"cycA\", \"abrB\", \"zupT\", \"ansP\", \n",
    "    \"wbbK\", \"wzzB\", \"murD\", \"rodZ\", \"mutL\", \"yibX\", \"def\", \"lspA\", \"cbrC\", \"sgcX\", \n",
    "    \"abpA\", \"insI2\", \"intZ\", \"rbsB\", \"rpe\", \"tar\", \"rnlA\", \"hyfF\", \"rcsA\", \"cryptic\", \n",
    "    \"ychO\", \"pgpB\", \"gyrA\", \"alaA\", \"yciQ\", \"ppdC\", \"ynaE\", \"oppA\", \"acnB\", \"gltT\", \n",
    "    \"ffh\", \"mntH\", \"ydhQ\", \"fliC\", \"rrlE\", \"nadK\", \"bax\", \"rfbC\", \"rbn\", \"radA\", \"glf\", \n",
    "    \"trpD\", \"rrsB\", \"rtcR\", \"gabP\", \"entF\", \"yqiI\", \"idi\", \"gltW\", \"rrlG\", \"dgcQ\", \"rrsA\", \n",
    "    \"ynjF\", \"aldA\", \"rrlD\", \"dnaK\", \"yohC\", \"nanY\", \"ynfD\", \"wbbH\", \"puuB\", \"fhuA\", \"zapA\", \n",
    "    \"rsmJ\", \"efeU\", \"narQ\", \"rrsG\", \"fliD\", \"purR\", \"appY\", \"ftnB\", \"folX\", \"argH\", \"nrdA\", \n",
    "    \"mdtF\", \"gpr\", \"ycjG\", \"deaD\", \"appC\", \"cyoA\", \"metA\", \"ymdF\", \"lolC\", \"rho\", \"ssrA\", \n",
    "    \"yibJ\", \"intA\", \"cycA\", \"abrB\", \"zupT\", \"ansP\", \"wbbK\", \"wzzB\", \"murD\", \"rodZ\", \"mutL\", \n",
    "    \"yibX\", \"def\", \"lspA\", \"cbrC\", \"sgcX\", \"abpA\", \"insI2\", \"intZ\", \"rbsB\", \"rpe\", \"tar\", \n",
    "    \"rnlA\", \"hyfF\", \"rcsA\", \"cryptic\", \"ychO\", \"pgpB\", \"gyrA\", \"alaA\", \"yciQ\", \"ppdC\", \n",
    "    \"ynaE\", \"oppA\", \"acnB\", \"gltT\", \"ffh\", \"mntH\", \"ydhQ\", \"fliC\", \"rrlE\", \"nadK\", \"bax\", \n",
    "    \"rfbC\", \"rbn\", \"radA\", \"glf\", \"trpD\", \"rrsB\", \"rtcR\", \"gabP\", \"entF\", \"yqiI\", \"idi\", \n",
    "    \"gltW\", \"rrlG\", \"dgcQ\", \"rrsA\", \"ynjF\", \"aldA\", \"rrlD\", \"dnaK\", \"yohC\", \"nanY\", \"ynfD\", \n",
    "    \"wbbH\", \"puuB\", \"fhuA\", \"zapA\", \"rsmJ\", \"efeU\", \"narQ\", \"rrsG\", \"fliD\", \"purR\", \"appY\", \n",
    "    \"ftnB\", \"folX\", \"argH\", \"nrdA\", \"mdtF\", \"gpr\", \"ycjG\", \"deaD\", \"appC\", \"cyoA\", \"metA\", \n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'abpA',\n",
       " 'abrB',\n",
       " 'acnB',\n",
       " 'alaA',\n",
       " 'aldA',\n",
       " 'ansP',\n",
       " 'appC',\n",
       " 'appY',\n",
       " 'argH',\n",
       " 'bax',\n",
       " 'cbrC',\n",
       " 'cryptic',\n",
       " 'cycA',\n",
       " 'cyoA',\n",
       " 'deaD',\n",
       " 'def',\n",
       " 'dgcQ',\n",
       " 'dnaK',\n",
       " 'efeU',\n",
       " 'entF',\n",
       " 'ffh',\n",
       " 'fhuA',\n",
       " 'fliC',\n",
       " 'fliD',\n",
       " 'folX',\n",
       " 'ftnB',\n",
       " 'gabP',\n",
       " 'glf',\n",
       " 'gltT',\n",
       " 'gltW',\n",
       " 'gpr',\n",
       " 'gyrA',\n",
       " 'hyfF',\n",
       " 'idi',\n",
       " 'insI2',\n",
       " 'intA',\n",
       " 'intZ',\n",
       " 'lolC',\n",
       " 'lspA',\n",
       " 'mdtF',\n",
       " 'metA',\n",
       " 'mntH',\n",
       " 'murD',\n",
       " 'mutL',\n",
       " 'nadK',\n",
       " 'nanY',\n",
       " 'narQ',\n",
       " 'nrdA',\n",
       " 'oppA',\n",
       " 'pgpB',\n",
       " 'ppdC',\n",
       " 'purR',\n",
       " 'puuB',\n",
       " 'radA',\n",
       " 'rbn',\n",
       " 'rbsB',\n",
       " 'rcsA',\n",
       " 'rfbC',\n",
       " 'rho',\n",
       " 'rnlA',\n",
       " 'rodZ',\n",
       " 'rpe',\n",
       " 'rrlD',\n",
       " 'rrlE',\n",
       " 'rrlG',\n",
       " 'rrsA',\n",
       " 'rrsB',\n",
       " 'rrsG',\n",
       " 'rsmJ',\n",
       " 'rtcR',\n",
       " 'sgcX',\n",
       " 'ssrA',\n",
       " 'tar',\n",
       " 'trpD',\n",
       " 'wbbH',\n",
       " 'wbbK',\n",
       " 'wzzB',\n",
       " 'ychO',\n",
       " 'yciQ',\n",
       " 'ycjG',\n",
       " 'ydhQ',\n",
       " 'yibJ',\n",
       " 'yibX',\n",
       " 'ymdF',\n",
       " 'ynaE',\n",
       " 'ynfD',\n",
       " 'ynjF',\n",
       " 'yohC',\n",
       " 'yqiI',\n",
       " 'zapA',\n",
       " 'zupT'}"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "set(topVarGenes) - set(Data.index)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['column 1', 'Gene products (complexes, polypeptides, and RNAs)',\n",
       "       'GO terms (biological process)', 'Pathways of gene',\n",
       "       'Binding sites upstream of gene'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Data.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "Data.columns = [\"Genes\", \"Products\", \"Biological Process\", \"Pathways\", \"Binding Sites\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/alec/miniconda3/envs/dCasRx/lib/python3.12/site-packages/IPython/core/formatters.py:347: FutureWarning: Index.format is deprecated and will be removed in a future version. Convert using index.astype(str) or index.map(formatter) instead.\n",
      "  return method()\n",
      "/Users/alec/miniconda3/envs/dCasRx/lib/python3.12/site-packages/IPython/core/formatters.py:347: FutureWarning: RangeIndex.format is deprecated and will be removed in a future version. Convert using index.astype(str) or index.map(formatter) instead.\n",
      "  return method()\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Genes</th>\n",
       "      <th>Products</th>\n",
       "      <th>Biological Process</th>\n",
       "      <th>Pathways</th>\n",
       "      <th>Binding Sites</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ymdF</td>\n",
       "      <td>stress-induced bacterial acidophilic repeat mo...</td>\n",
       "      <td>cell adhesion involved in single-species biofi...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>lolC</td>\n",
       "      <td>lipoprotein release complex</td>\n",
       "      <td>protein localization to outer membrane // lipo...</td>\n",
       "      <td>lipoprotein posttranslational modification (Gr...</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>rho</td>\n",
       "      <td>transcription termination factor Rho</td>\n",
       "      <td>DNA-templated transcription termination</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ssrA</td>\n",
       "      <td>tmRNA</td>\n",
       "      <td>rescue of stalled ribosome // trans-translatio...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>yibJ</td>\n",
       "      <td>putative RHS domain-containing protein YibJ</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Genes                                           Products   \n",
       "0  ymdF  stress-induced bacterial acidophilic repeat mo...  \\\n",
       "1  lolC                        lipoprotein release complex   \n",
       "2   rho               transcription termination factor Rho   \n",
       "3  ssrA                                              tmRNA   \n",
       "4  yibJ        putative RHS domain-containing protein YibJ   \n",
       "\n",
       "                                  Biological Process   \n",
       "0  cell adhesion involved in single-species biofi...  \\\n",
       "1  protein localization to outer membrane // lipo...   \n",
       "2            DNA-templated transcription termination   \n",
       "3  rescue of stalled ribosome // trans-translatio...   \n",
       "4                                                NaN   \n",
       "\n",
       "                                            Pathways Binding Sites  \n",
       "0                                                NaN           NaN  \n",
       "1  lipoprotein posttranslational modification (Gr...           NaN  \n",
       "2                                                NaN           NaN  \n",
       "3                                                NaN           NaN  \n",
       "4                                                NaN           NaN  "
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [],
   "source": [
    "Data[\"Biological Process\"] = Data[\"Biological Process\"].str.replace(\" //\" , \",\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {},
   "outputs": [],
   "source": [
    "for col in Data.columns:\n",
    "    Data[col] = Data[col].str.replace(\" //\", \",\")\n",
    "    Data[col] = Data[col].str.wrap(30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\begin{tabular}{rlllll}\n",
      "\\hline\n",
      "     & Genes   & Products                       & Biological Process            & Pathways                & Binding Sites   \\\\\n",
      "\\hline\n",
      "   0 & ymdF    & stress-induced bacterial\n",
      "acidophilic repeat motifs-\n",
      "containing protein YmdF                                & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation                               & nan                     & nan             \\\\\n",
      "   1 & lolC    & lipoprotein release complex    & protein localization to outer\n",
      "membrane, lipoprotein\n",
      "transport, lipoprotein\n",
      "localization to outer membrane                               & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria)                         & nan             \\\\\n",
      "   2 & rho     & transcription termination\n",
      "factor Rho                                & DNA-templated transcription\n",
      "termination                               & nan                     & nan             \\\\\n",
      "   3 & ssrA    & tmRNA                          & rescue of stalled ribosome,\n",
      "trans-translation-dependent\n",
      "protein tagging, trans-\n",
      "translation                               & nan                     & nan             \\\\\n",
      "   4 & yibJ    & putative RHS domain-containing\n",
      "protein YibJ                                & nan                           & nan                     & nan             \\\\\n",
      "   5 & intA    & CP4-57 prophage; integrase\n",
      "IntA                                & viral genome integration into\n",
      "host DNA, symbiont entry into\n",
      "host cell, establishment of\n",
      "integrated proviral latency,\n",
      "DNA integration, DNA\n",
      "recombination, provirus\n",
      "excision                               & nan                     & nan             \\\\\n",
      "   6 & cycA    & D-\n",
      "serine/alanine/glycine:H\\ensuremath{<}sup\\ensuremath{>}+\n",
      "\\ensuremath{<}/sup\\ensuremath{>}symporter                                & transmembrane transport, amino\n",
      "acid transport, D-alanine\n",
      "transmembrane transport,\n",
      "L-alanine transport, beta-\n",
      "alanine transport, glycine\n",
      "transport, D-serine\n",
      "transmembrane transport                               & nan                     & nan             \\\\\n",
      "   7 & abrB    & putative regulator             & regulation of gene expression & nan                     & nan             \\\\\n",
      "   8 & zupT    & divalent metal ion transporter\n",
      "ZupT                                & iron ion transport, zinc ion\n",
      "transport, cobalt ion\n",
      "transport, zinc ion import\n",
      "across plasma membrane, iron\n",
      "ion import across plasma\n",
      "membrane, zinc ion\n",
      "transmembrane transport,\n",
      "transmembrane transport, metal\n",
      "ion transport                               & nan                     & nan             \\\\\n",
      "   9 & ansP    & L-asparagine transporter       & transmembrane transport, amino\n",
      "acid transport                               & nan                     & nan             \\\\\n",
      "  10 & wbbK    & putative glycosyltransferase\n",
      "WbbK                                & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      "  11 & wzzB    & O-antigen co-polymerase        & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      "  12 & murD    & UDP-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-acetylmuramoyl-L-\n",
      "alanine\\&mdash;D-glutamate\n",
      "ligase                                & cell wall organization,\n",
      "biosynthetic process, cell\n",
      "division, regulation of cell\n",
      "shape, peptidoglycan\n",
      "biosynthetic process                               & peptidoglycan biosynthesis I\n",
      "(\\ensuremath{<}I\\ensuremath{>}meso\\ensuremath{<}/I\\ensuremath{>}-diaminopimelate\n",
      "containing),\n",
      "UDP-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-acetylmuramoyl-\n",
      "pentapeptide biosynthesis I\n",
      "(\\ensuremath{<}i\\ensuremath{>}meso\\ensuremath{<}/i\\ensuremath{>}-diaminopimelate\n",
      "containing)                         & LexA DNA-binding-site 3 bases\n",
      "downstream of ftsLp2, MraZ\n",
      "DNA-binding-site 18 bases\n",
      "downstream of mraZp, MraZ DNA-\n",
      "binding-site 8 bases\n",
      "downstream of mraZp, MraZ DNA-\n",
      "binding-site 3 bases upstream\n",
      "of mraZp, PdhR DNA-binding-\n",
      "site 42 bases upstream of\n",
      "mraZp                 \\\\\n",
      "  13 & rodZ    & cytoskeleton protein RodZ      & peptidoglycan biosynthetic\n",
      "process, FtsZ-dependent\n",
      "cytokinesis, regulation of\n",
      "cell shape                               & nan                     & nan             \\\\\n",
      "  14 & mutL    & methyl-directed mismatch\n",
      "repair system                                & nucleotide-excision repair,\n",
      "DNA duplex unwinding,\n",
      "regulation of DNA\n",
      "recombination, DNA repair,\n",
      "mismatch repair involved in\n",
      "maintenance of fidelity\n",
      "involved in DNA-dependent DNA\n",
      "replication, mismatch repair                               & nan                     & nan             \\\\\n",
      "  15 & yibX    & protein YibX-S                 & nan                           & nan                     & nan             \\\\\n",
      "  16 & def     & peptide deformylase            & translation, co-translational\n",
      "protein modification,\n",
      "N-terminal protein amino acid\n",
      "modification                               & nan                     & nan             \\\\\n",
      "  17 & lspA    & lipoprotein signal peptidase   & proteolysis, lipoprotein\n",
      "biosynthetic process via\n",
      "signal peptide cleavage                               & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria)                         & nan             \\\\\n",
      "  18 & cbrC    & PF03691 family colicin E2\n",
      "tolerance protein CbrC                                & bacteriocin immunity          & nan                     & nan             \\\\\n",
      "  19 & sgcX    & KpLE2 phage-like element;\n",
      "putative endoglucanase with\n",
      "Zn-dependent exopeptidase\n",
      "domain                                & proteolysis                   & nan                     & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site regulating\n",
      "sgcXBCQAER, SlyA DNA-binding-\n",
      "site regulating sgcXBCQAER\n",
      "(BS0-8641), SlyA DNA-binding-\n",
      "site regulating sgcXBCQAER\n",
      "(BS0-8639)                 \\\\\n",
      "  20 & abpA    & CP4-57 prophage; anti-\n",
      "bacteriophage protein                                & defense response to virus     & nan                     & nan             \\\\\n",
      "  21 & intZ    & CPZ-55 prophage; putative\n",
      "phage integrase IntZ                                & viral genome integration into\n",
      "host DNA, establishment of\n",
      "integrated proviral latency,\n",
      "symbiont entry into host cell,\n",
      "DNA integration, DNA\n",
      "recombination                               & nan                     & nan             \\\\\n",
      "  22 & rbsB    & ribose ABC transporter         & chemotaxis, D-ribose\n",
      "transmembrane transport,\n",
      "positive chemotaxis                               & nan                     & RbsR DNA-binding-site 8.5\n",
      "bases downstream of rbsDp,\n",
      "CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 62.5 bases\n",
      "upstream of rbsDp                 \\\\\n",
      "  23 & rpe     & ribulose-phosphate 3-epimerase & pentose-phosphate shunt,\n",
      "carbohydrate metabolic\n",
      "process, pentose catabolic\n",
      "process, pentose-phosphate\n",
      "shunt, non-oxidative branch                               & pentose phosphate pathway,\n",
      "pentose phosphate pathway\n",
      "(non-oxidative branch) I                         & nan             \\\\\n",
      "  24 & tar     & chemotaxis signaling complex\n",
      "core unit - aspartate sensing                                & positive regulation of post-\n",
      "translational protein\n",
      "modification, cellular\n",
      "response to amino acid\n",
      "stimulus, regulation of\n",
      "chemotaxis, detection of\n",
      "chemical stimulus, signal\n",
      "complex assembly, signal\n",
      "transduction, protein\n",
      "homooligomerization,\n",
      "chemotaxis                               & nan                     & Fnr DNA-binding-site\n",
      "regulating tarp                 \\\\\n",
      "  25 & rnlA    & CP4-57 prophage; RNase LS,\n",
      "toxin of the RnlAB toxin-\n",
      "antitoxin system, RnlA-RnlB\n",
      "toxin-antitoxin complex                                & regulation of DNA-templated\n",
      "transcription, single-species\n",
      "biofilm formation, regulation\n",
      "of growth, defense response to\n",
      "virus, mRNA catabolic process                               & nan                     & IscR DNA-binding-site 29.5\n",
      "bases upstream of rnlAp2                 \\\\\n",
      "  26 & hyfF    & hydrogenase 4                  & anaerobic electron transport\n",
      "chain, formate oxidation,\n",
      "anaerobic respiration, glucose\n",
      "catabolic process, DNA damage\n",
      "response                               & nan                     & Fnr DNA-binding-site 105.5\n",
      "bases upstream of hyfAp, HyfR\n",
      "DNA-binding-site 129.5 bases\n",
      "upstream of hyfAp, FhlA-\n",
      "formate DNA-binding-site 130\n",
      "bases upstream of hyfAp, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 160.5 bases\n",
      "upstream of hyfAp                 \\\\\n",
      "  27 & rcsA    & RcsAB DNA-binding\n",
      "transcriptional dual regulator                                & positive regulation of DNA-\n",
      "templated transcription,\n",
      "negative regulation of DNA-\n",
      "templated transcription,\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility, regulation of\n",
      "capsule organization,\n",
      "regulation of DNA-templated\n",
      "transcription, DNA-templated\n",
      "transcription                               & nan                     & H-NS DNA-binding-site\n",
      "regulating rcsAp, RcsAB DNA-\n",
      "binding-site 125 bases\n",
      "upstream of rcsAp, GadE DNA-\n",
      "binding-site regulating rcsAp,\n",
      "RcsAB DNA-binding-site 172\n",
      "bases upstream of rcsAp                 \\\\\n",
      "  28 & ychO    & putative invasin YchO          & nan                           & nan                     & Fnr DNA-binding-site\n",
      "regulating ychOp                 \\\\\n",
      "  29 & pgpB    & phosphatidylglycerophosphatase\n",
      "B                                & phospholipid catabolic\n",
      "process, phosphatidylglycerol\n",
      "biosynthetic process,\n",
      "peptidoglycan biosynthetic\n",
      "process, glycerophospholipid\n",
      "biosynthetic process                               & superpathway of phospholipid\n",
      "biosynthesis III (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), cardiolipin\n",
      "biosynthesis III, cardiolipin\n",
      "biosynthesis I, polyisoprenoid\n",
      "biosynthesis (\\ensuremath{<}i\\ensuremath{>}E. coli\\ensuremath{<}/i\\ensuremath{>}),\n",
      "di-\\ensuremath{<}i\\ensuremath{>}trans\\ensuremath{<}/i\\ensuremath{>},poly-\n",
      "\\ensuremath{<}i\\ensuremath{>}cis\\ensuremath{<}/i\\ensuremath{>}-undecaprenyl\n",
      "phosphate biosynthesis                         & nan             \\\\\n",
      "  30 & gyrA    & DNA gyrase                     & chromosome organization,\n",
      "response to xenobiotic\n",
      "stimulus, negative regulation\n",
      "of DNA-templated DNA\n",
      "replication, DNA-templated DNA\n",
      "replication, DNA topological\n",
      "change, response to\n",
      "antibiotic, DNA-templated\n",
      "transcription                               & nan                     & Fis DNA-binding-site 42.5\n",
      "bases upstream of gyrAp, CspA\n",
      "DNA-binding-site 108 bases\n",
      "upstream of gyrAp, CspA DNA-\n",
      "binding-site 88 bases upstream\n",
      "of gyrAp, CspA DNA-binding-\n",
      "site 60 bases upstream of\n",
      "gyrAp                 \\\\\n",
      "  31 & alaA    & glutamate\\&mdash;pyruvate\n",
      "aminotransferase AlaA                                & amino acid biosynthetic\n",
      "process, biosynthetic process,\n",
      "alanine biosynthetic process,\n",
      "D-alanine biosynthetic\n",
      "process, DNA damage response,\n",
      "response to antibiotic,\n",
      "L-alanine biosynthetic process\n",
      "from pyruvate                               & superpathway of L-alanine\n",
      "biosynthesis, L-alanine\n",
      "biosynthesis II                         & nan             \\\\\n",
      "  32 & yciQ    & DUF2207 domain-containing\n",
      "protein YciQ                                & nan                           & nan                     & nan             \\\\\n",
      "  33 & ppdC    & prepilin-type N-terminal\n",
      "cleavage/methylation domain-\n",
      "containing protein PpdC                                & nan                           & nan                     & nan             \\\\\n",
      "  34 & ynaE    & Rac prophage; uncharacterized\n",
      "protein YnaE                                & response to cold              & nan                     & GlaR DNA-binding-site\n",
      "regulating ynaE                 \\\\\n",
      "  35 & oppA    & oligopeptide ABC transporter   & oligopeptide import across\n",
      "plasma membrane, transmembrane\n",
      "transport, peptide transport,\n",
      "protein transport, chaperone-\n",
      "mediated protein folding,\n",
      "response to heat, oligopeptide\n",
      "transport                               & nan                     & DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site 136 bases\n",
      "upstream of oppAp, Lrp DNA-\n",
      "binding-site 1818 bases\n",
      "upstream of oppAp, ArcA-Pasp54\n",
      "DNA-binding-site regulating\n",
      "oppAp                 \\\\\n",
      "  36 & acnB    & bifunctional aconitate\n",
      "hydratase B/2-methylisocitrate\n",
      "dehydratase                                & glyoxylate cycle,\n",
      "tricarboxylic acid cycle,\n",
      "regulation of translation,\n",
      "propionate catabolic process,\n",
      "2-methylcitrate cycle                               & superpathway of glycolysis,\n",
      "pyruvate dehydrogenase, TCA,\n",
      "and glyoxylate bypass,\n",
      "superpathway of glyoxylate\n",
      "bypass and TCA, glyoxylate\n",
      "cycle, TCA cycle I\n",
      "(prokaryotic), mixed acid\n",
      "fermentation, 2-methylcitrate\n",
      "cycle I                         & ArcA-Pasp54 DNA-binding-site\n",
      "48 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "57 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "119 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "113 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "139 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "143 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "170 bases upstream of acnBp1,\n",
      "Fis DNA-binding-site 29.5\n",
      "bases upstream of acnBp1, Fis\n",
      "DNA-binding-site 73.5 bases\n",
      "upstream of acnBp1, Fis DNA-\n",
      "binding-site 171.5 bases\n",
      "upstream of acnBp1, Cra DNA-\n",
      "binding-site regulating\n",
      "acnBp1, CRP-cyclic-AMP DNA-\n",
      "binding transcriptional dual\n",
      "regulator DNA-binding-site\n",
      "37.5 bases upstream of acnBp1                 \\\\\n",
      "  37 & gltT    & tRNA-Glu(UUC)                  & nan                           & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging                         & Fis DNA-binding-site 143 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 53 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "H-NS DNA-binding-site 50 bases\n",
      "upstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 75 bases downstream of\n",
      "rrsBp, Lrp-leu DNA-binding-\n",
      "site 269 bases downstream of\n",
      "rrsBp, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsBp2, NhaR-\n",
      "Na+ DNA-binding-site\n",
      "regulating rrsBp1, Lrp-leu\n",
      "DNA-binding-site 269 bases\n",
      "downstream of rrsBp, Lrp-leu\n",
      "DNA-binding-site 75 bases\n",
      "downstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, H-NS DNA-binding-\n",
      "site 50 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsBp, Fis\n",
      "DNA-binding-site 71 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsBp                 \\\\\n",
      "  38 & ffh     & Signal Recognition Particle    & SRP-dependent cotranslational\n",
      "protein targeting to membrane,\n",
      "protein targeting to membrane                               & nan                     & nan             \\\\\n",
      "  39 & mntH    & Mn(2+):H(+) symporter          & iron ion transmembrane\n",
      "transport, proton\n",
      "transmembrane transport,\n",
      "cadmium ion transmembrane\n",
      "transport, manganese ion\n",
      "transport, iron ion transport,\n",
      "cobalt ion transport, cadmium\n",
      "ion transport, monoatomic ion\n",
      "transport, metal ion\n",
      "transport, manganese import\n",
      "into cell, cellular response\n",
      "to oxidative stress                               & nan                     & OxyR DNA-binding-site 76 bases\n",
      "upstream of mntHp, OxyR DNA-\n",
      "binding-site 54 bases upstream\n",
      "of mntHp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 31.5\n",
      "bases upstream of mntHp, MntR\n",
      "DNA-binding-site 5.5 bases\n",
      "downstream of mntHp                 \\\\\n",
      "  40 & ydhQ    & putative adhesin-related\n",
      "protein YdhQ                                & DNA damage response           & nan                     & nan             \\\\\n",
      "  41 & fliC    & flagellum                      & nan                           & nan                     & MatA DNA-binding-site\n",
      "regulating fliCp, GadE DNA-\n",
      "binding-site regulating fliCp                 \\\\\n",
      "  42 & rrlE    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 170 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 117 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 101 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 72 bases upstream\n",
      "of rrsEp3, NhaR-Na+ DNA-\n",
      "binding-site regulating\n",
      "rrsEp2, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsEp1, Fis\n",
      "DNA-binding-site 72 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 101 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 117 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 170 bases\n",
      "upstream of rrsEp3                 \\\\\n",
      "  43 & nadK    & NAD kinase                     & NAD metabolic process, NADP\n",
      "biosynthetic process                               & NAD phosphorylation and\n",
      "dephosphorylation                         & nan             \\\\\n",
      "  44 & bax     & putative glycoside hydrolase\n",
      "Bax                                & nan                           & nan                     & nan             \\\\\n",
      "  45 & rfbC    & dTDP-4-dehydrorhamnose\n",
      "3,5-epimerase                                & dTDP-rhamnose biosynthetic\n",
      "process, extracellular\n",
      "polysaccharide biosynthetic\n",
      "process, lipopolysaccharide\n",
      "biosynthetic process, O\n",
      "antigen biosynthetic process,\n",
      "response to UV, response to\n",
      "antibiotic, DNA damage\n",
      "response                               & \\ensuremath{<}i\\ensuremath{>}O\\ensuremath{<}/i\\ensuremath{>}-antigen building\n",
      "blocks biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), dTDP-L-rhamnose\n",
      "biosynthesis                         & nan             \\\\\n",
      "  46 & radA    & DNA recombination protein      & response to radiation, DNA\n",
      "repair, recombinational repair                               & nan                     & nan             \\\\\n",
      "  47 & glf     & UDP-galactopyranose mutase     & lipopolysaccharide\n",
      "biosynthetic process                               & \\ensuremath{<}i\\ensuremath{>}O\\ensuremath{<}/i\\ensuremath{>}-antigen building\n",
      "blocks biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>})                         & nan             \\\\\n",
      "  48 & trpD    & anthranilate synthase subunit\n",
      "TrpE, anthranilate synthase\n",
      "subunit TrpD                                & phenazine biosynthetic\n",
      "process, glutamine metabolic\n",
      "process, tryptophan\n",
      "biosynthetic process                               & L-glutamine degradation I,\n",
      "L-glutamate biosynthesis I,\n",
      "ammonia assimilation cycle\n",
      "III, superpathway of\n",
      "chorismate metabolism,\n",
      "superpathway of aromatic amino\n",
      "acid biosynthesis,\n",
      "superpathway of L-tryptophan\n",
      "biosynthesis, L-tryptophan\n",
      "biosynthesis                         & TrpR-trp DNA-binding-site 11.5\n",
      "bases upstream of trpLp, TrpR-\n",
      "trp DNA-binding-site 3.5 bases\n",
      "upstream of trpLp, TrpR-trp\n",
      "DNA-binding-site 19.5 bases\n",
      "upstream of trpLp                 \\\\\n",
      "  49 & rrsB    & 16S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 143 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 53 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "H-NS DNA-binding-site 50 bases\n",
      "upstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 75 bases downstream of\n",
      "rrsBp, Lrp-leu DNA-binding-\n",
      "site 269 bases downstream of\n",
      "rrsBp, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsBp2, NhaR-\n",
      "Na+ DNA-binding-site\n",
      "regulating rrsBp1, Lrp-leu\n",
      "DNA-binding-site 269 bases\n",
      "downstream of rrsBp, Lrp-leu\n",
      "DNA-binding-site 75 bases\n",
      "downstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, H-NS DNA-binding-\n",
      "site 50 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsBp, Fis\n",
      "DNA-binding-site 71 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsBp                 \\\\\n",
      "  50 & rtcR    & DNA-binding transcriptional\n",
      "activator RtcR                                & regulation of DNA-templated\n",
      "transcription, phosphorelay\n",
      "signal transduction system,\n",
      "positive regulation of DNA-\n",
      "templated transcription                               & nan                     & nan             \\\\\n",
      "  51 & gabP    & 4-aminobutanoate:H\\ensuremath{<}sup\\ensuremath{>}+\\ensuremath{<}/sup\\ensuremath{>}\n",
      "symporter                                & transmembrane transport,\n",
      "gamma-aminobutyric acid\n",
      "catabolic process, DNA damage\n",
      "response, amino acid\n",
      "transport, cellular response\n",
      "to nitrogen starvation,\n",
      "response to carbon starvation,\n",
      "gamma-aminobutyric acid\n",
      "transport                               & nan                     & Nac DNA-binding-site 44 bases\n",
      "upstream of gabDp, GlaR DNA-\n",
      "binding-site 32 bases upstream\n",
      "of csiDp, GlaR DNA-binding-\n",
      "site 13 bases upstream of\n",
      "csiDp, CRP-cyclic-AMP DNA-\n",
      "binding transcriptional dual\n",
      "regulator DNA-binding-site\n",
      "68.5 bases upstream of csiDp,\n",
      "Lrp-leu DNA-binding-site\n",
      "regulating csiDp, H-NS DNA-\n",
      "binding-site regulating csiDp,\n",
      "Nac DNA-binding-site 44 bases\n",
      "upstream of gabDp                 \\\\\n",
      "  52 & entF    & seryl-[EntF peptidyl-carrier\n",
      "protein]                                & secondary metabolite\n",
      "biosynthetic process,\n",
      "biosynthetic process, amino\n",
      "acid activation for\n",
      "nonribosomal peptide\n",
      "biosynthetic process,\n",
      "enterobactin biosynthetic\n",
      "process                               & superpathway of chorismate\n",
      "metabolism, acyl carrier\n",
      "protein metabolism,\n",
      "enterobactin biosynthesis                         & H-NS DNA-binding-site 0.5\n",
      "bases upstream of fesp, DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "28 bases upstream of fesp,\n",
      "DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site 9 bases upstream\n",
      "of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 15 bases\n",
      "upstream of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 2 bases\n",
      "upstream of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 21 bases\n",
      "upstream of fesp, Fnr DNA-\n",
      "binding-site 143.5 bases\n",
      "upstream of fesp                 \\\\\n",
      "  53 & yqiI    & putative fimbrial protein YqiI & cell adhesion, response to\n",
      "methylglyoxal                               & nan                     & nan             \\\\\n",
      "  54 & idi     & isopentenyl-diphosphate\n",
      "\\&Delta;-isomerase                                & dimethylallyl diphosphate\n",
      "biosynthetic process, DNA\n",
      "damage response, isoprenoid\n",
      "biosynthetic process                               & methylerythritol phosphate\n",
      "pathway I                         & nan             \\\\\n",
      "  55 & gltW    & tRNA-Glu(UUC)                  & nan                           & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging                         & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      "  56 & rrlG    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      "  57 & dgcQ    & diguanylate cyclase DgcQ       & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation, cellulose\n",
      "biosynthetic process, negative\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility                               & nan                     & nan             \\\\\n",
      "  58 & rrsA    & ribosome                       & nan                           & nan                     & Fis DNA-binding-site 133 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 111 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsAp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsAp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsAp, Lrp-\n",
      "leu DNA-binding-site 73 bases\n",
      "upstream of rrsAp, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsAp2, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsAp1, Lrp-\n",
      "leu DNA-binding-site 73 bases\n",
      "upstream of rrsAp, Lrp-leu\n",
      "DNA-binding-site 53 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsAp, Fis DNA-binding-site\n",
      "91 bases upstream of rrsAp,\n",
      "Fis DNA-binding-site 111 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 133 bases\n",
      "upstream of rrsAp                 \\\\\n",
      "  59 & ynjF    & CDP-alcohol\n",
      "phosphatidyltransferase\n",
      "domain-containing protein YnjF                                & phospholipid biosynthetic\n",
      "process                               & nan                     & nan             \\\\\n",
      "  60 & aldA    & aldehyde dehydrogenase A, NAD-\n",
      "linked                                & gamma-aminobutyric acid\n",
      "catabolic process, fucose\n",
      "catabolic process, rhamnose\n",
      "catabolic process, L-fucose\n",
      "catabolic process                               & superpathway of fucose and\n",
      "rhamnose degradation,\n",
      "methylglyoxal degradation IV,\n",
      "superpathway of methylglyoxal\n",
      "degradation, L-lactaldehyde\n",
      "degradation (aerobic),\n",
      "superpathway of glycol\n",
      "metabolism and degradation,\n",
      "ethylene glycol degradation,\n",
      "D-arabinose degradation II                         & SoxS DNA-binding-site 161\n",
      "bases upstream of aldAp, Rob\n",
      "DNA-binding-site 161 bases\n",
      "upstream of aldAp, MarA DNA-\n",
      "binding-site 161 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 157 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 5 bases\n",
      "upstream of aldAp, DnaA-ATP\n",
      "DNA-binding-site 7.5 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 2 bases\n",
      "downstream of aldAp, ArcA-\n",
      "Pasp54 DNA-binding-site 13\n",
      "bases downstream of aldAp,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "137 bases upstream of aldAp,\n",
      "Fnr DNA-binding-site 10.5\n",
      "bases upstream of aldAp, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 59.5 bases\n",
      "upstream of aldAp                 \\\\\n",
      "  61 & rrlD    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 174 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 144 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 126 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsDp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsDp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsDp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsDp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsDp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 126 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 144 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 174 bases\n",
      "upstream of rrsDp                 \\\\\n",
      "  62 & dnaK    & chaperone protein DnaK         & protein refolding, protein-\n",
      "containing complex assembly,\n",
      "negative regulation of DNA-\n",
      "templated transcription, DNA\n",
      "replication, stress response\n",
      "to copper ion, protein\n",
      "folding, chaperone cofactor-\n",
      "dependent protein refolding,\n",
      "cellular response to unfolded\n",
      "protein, protein unfolding,\n",
      "response to heat                               & nan                     & nan             \\\\\n",
      "  63 & yohC    & putative inner membrane\n",
      "protein                                & DNA damage response           & nan                     & nan             \\\\\n",
      "  64 & nanY    & KpLE2 phage-like element; 2,7-\n",
      "anhydro-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-\n",
      "acetylneuraminate hydratase                                & DNA damage response,\n",
      "N-acetylneuraminate catabolic\n",
      "process                               & nan                     & NanR DNA-binding-site 376.5\n",
      "bases upstream of yjhBp2, NanR\n",
      "DNA-binding-site 384.5 bases\n",
      "upstream of yjhBp2, NanR DNA-\n",
      "binding-site 393.5 bases\n",
      "upstream of yjhBp2                 \\\\\n",
      "  65 & ynfD    & DUF1161 domain-containing\n",
      "protein YnfD                                & nan                           & nan                     & nan             \\\\\n",
      "  66 & wbbH    & putative O-antigen polymerase  & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      "  67 & puuB    & \\&gamma;-glutamylputrescine\n",
      "oxidase                                & putrescine catabolic process  & superpathway of L-arginine and\n",
      "L-ornithine degradation,\n",
      "superpathway of ornithine\n",
      "degradation, superpathway of\n",
      "L-arginine, putrescine, and\n",
      "4-aminobutanoate degradation,\n",
      "putrescine degradation II                         & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 63.5 bases\n",
      "downstream of puuDp, PuuR DNA-\n",
      "binding-site 161.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 126.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 84.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 11.5 bases\n",
      "downstream of puuAp, ArcA-\n",
      "Pasp54 DNA-binding-site 82\n",
      "bases downstream of puuDp,\n",
      "PtrR-gln DNA-binding-site\n",
      "regulating puuCp                 \\\\\n",
      "  68 & fhuA    & ferrichrome outer membrane\n",
      "transport complex                                & iron ion transport,\n",
      "intracellular iron ion\n",
      "homeostasis, siderophore-\n",
      "dependent iron import into\n",
      "cell, siderophore transport,\n",
      "siderophore transmembrane\n",
      "transport                               & nan                     & DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site regulating\n",
      "fhuACDB (BS0-10960), DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "regulating fhuACDB\n",
      "(BS0-10871), DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 3 bases\n",
      "upstream of fhuAp                 \\\\\n",
      "  69 & zapA    & cell division protein ZapA     & cell septum assembly, septin\n",
      "ring assembly, division septum\n",
      "assembly, FtsZ-dependent\n",
      "cytokinesis, cell division                               & nan                     & nan             \\\\\n",
      "  70 & rsmJ    & 16S rRNA m\\ensuremath{<}sup\\ensuremath{>}2\\ensuremath{<}/sup\\ensuremath{>}G1516\n",
      "methyltransferase                                & methylation, rRNA processing,\n",
      "rRNA methylation, rRNA base\n",
      "methylation                               & nan                     & nan             \\\\\n",
      "  71 & efeU    & inactive ferrous iron permease\n",
      "EfeU                                & iron ion transmembrane\n",
      "transport                               & nan                     & CpxR-phosphorylated DNA-\n",
      "binding-site 54.5 bases\n",
      "upstream of efeUp                 \\\\\n",
      "  72 & narQ    & sensor histidine kinase NarQ   & phosphorelay signal\n",
      "transduction system, signal\n",
      "transduction, nitrate\n",
      "assimilation, cellular\n",
      "response to nitrate, cellular\n",
      "response to nitrite                               & NarQ Two-Component Signal\n",
      "Transduction System, nitrate\n",
      "dependent                         & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 0.5 bases\n",
      "upstream of narQp                 \\\\\n",
      "  73 & rrsG    & 16S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      "  74 & fliD    & flagellum                      & cell adhesion, bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility                               & nan                     & FlhDC DNA-binding-site 55\n",
      "bases upstream of fliDp2                 \\\\\n",
      "  75 & purR    & DNA-binding transcriptional\n",
      "repressor PurR-hypoxanthine                                & regulation of DNA-templated\n",
      "transcription, purine\n",
      "nucleotide biosynthetic\n",
      "process, negative regulation\n",
      "of purine nucleotide\n",
      "biosynthetic process, negative\n",
      "regulation of DNA-templated\n",
      "transcription                               & nan                     & PurR-hypoxanthine DNA-binding-\n",
      "site 191.5 bases downstream of\n",
      "purRp, PurR-hypoxanthine DNA-\n",
      "binding-site 103.5 bases\n",
      "downstream of purRp, DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "49 bases upstream of purRp                 \\\\\n",
      "  76 & appY    & DLP12 prophage; DNA-binding\n",
      "transcriptional activator AppY                                & regulation of DNA-templated\n",
      "transcription, response to\n",
      "X-ray, cellular response to\n",
      "phosphate starvation, positive\n",
      "regulation of DNA-templated\n",
      "transcription                               & nan                     & ArcA-Pasp54 DNA-binding-site\n",
      "197 bases upstream of appYp2,\n",
      "DpiA-Pasp DNA-binding-site\n",
      "regulating appYp1, H-NS DNA-\n",
      "binding-site regulating appYp1                 \\\\\n",
      "  77 & ftnB    & putative ferritin-like protein & intracellular sequestering of\n",
      "iron ion                               & nan                     & CpxR-phosphorylated DNA-\n",
      "binding-site 61 bases upstream\n",
      "of ftnBp2                 \\\\\n",
      "  78 & folX    & dihydroneopterin triphosphate\n",
      "2'-epimerase                                & pteridine-containing compound\n",
      "biosynthetic process, folic\n",
      "acid-containing compound\n",
      "metabolic process                               & tetrahydromonapterin\n",
      "biosynthesis                         & nan             \\\\\n",
      "  79 & argH    & argininosuccinate lyase        & arginine biosynthetic process\n",
      "via ornithine, arginine\n",
      "biosynthetic process                               & superpathway of arginine and\n",
      "polyamine biosynthesis,\n",
      "L-arginine biosynthesis I (via\n",
      "L-ornithine)                         & ArgR-arg DNA-binding-site 6.5\n",
      "bases upstream of argEp1,\n",
      "ArgR-arg DNA-binding-site 16.5\n",
      "bases downstream of argEp1                 \\\\\n",
      "  80 & nrdA    & ribonucleoside-diphosphate\n",
      "reductase 1                                & protein folding,\n",
      "2'-deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "ribonucleoside diphosphate\n",
      "metabolic process,\n",
      "deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "nucleobase-containing small\n",
      "molecule interconversion                               & superpathway of pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), superpathway of\n",
      "pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis,\n",
      "pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis I,\n",
      "superpathway of adenosine\n",
      "nucleotides \\ensuremath{<}i\\ensuremath{>}de novo\\ensuremath{<}/i\\ensuremath{>}\n",
      "biosynthesis II, adenosine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II,\n",
      "superpathway of histidine,\n",
      "purine, and pyrimidine\n",
      "biosynthesis, superpathway of\n",
      "purine nucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II,\n",
      "superpathway of guanosine\n",
      "nucleotides \\ensuremath{<}i\\ensuremath{>}de novo\\ensuremath{<}/i\\ensuremath{>}\n",
      "biosynthesis II, guanosine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II                         & H-NS DNA-binding-site 415\n",
      "bases upstream of nrdAp, H-NS\n",
      "DNA-binding-site 439 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 334.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 281.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 244.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 14 bases\n",
      "upstream of nrdAp, NrdR\n",
      "transcriptional repressor DNA-\n",
      "binding-site 25.5 bases\n",
      "downstream of nrdAp, NrdR\n",
      "transcriptional repressor DNA-\n",
      "binding-site 7.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 36.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 48 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 156 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 163.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 190.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 218.5 bases\n",
      "upstream of nrdAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 136.5 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 128 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 199 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 269 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 323 bases\n",
      "upstream of nrdAp                 \\\\\n",
      "  81 & mdtF    & multidrug efflux pump MdtEF-\n",
      "TolC                                & xenobiotic transmembrane\n",
      "transport, transmembrane\n",
      "transport, xenobiotic\n",
      "transport, response to toxic\n",
      "substance, bile acid and bile\n",
      "salt transport, response to\n",
      "antibiotic                               & nan                     & FliZ DNA-binding-site 9.5\n",
      "bases downstream of gadEp2,\n",
      "GadW DNA-binding-site 574.5\n",
      "bases upstream of gadEp2, GadX\n",
      "DNA-binding-site 29.5 bases\n",
      "upstream of gadEp3, H-NS DNA-\n",
      "binding-site regulating\n",
      "gadEp2, GadX DNA-binding-site\n",
      "50.5 bases upstream of gadEp3,\n",
      "GadW DNA-binding-site 595.5\n",
      "bases upstream of gadEp2, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 19.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 30.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 63.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 198.5 bases\n",
      "upstream of gadEp2, EvgA-Pasp\n",
      "DNA-binding-site regulating\n",
      "gadEp2, YdeO DNA-binding-site\n",
      "regulating gadEp2                 \\\\\n",
      "  82 & gpr     & L-glyceraldehyde 3-phosphate\n",
      "reductase                                & DNA damage response,\n",
      "methylglyoxal catabolic\n",
      "process                               & superpathway of methylglyoxal\n",
      "degradation, methylglyoxal\n",
      "degradation III                         & nan             \\\\\n",
      "  83 & ycjG    & L-Ala-D/L-Glu epimerase        & cell wall organization, amino\n",
      "acid catabolic process,\n",
      "peptidoglycan turnover                               & muropeptide degradation & PgrR DNA-binding-site\n",
      "regulating ycjGp                 \\\\\n",
      "  84 & deaD    & ATP-dependent RNA helicase\n",
      "DeaD                                & response to cold, mRNA\n",
      "stabilization, response to\n",
      "radiation, positive regulation\n",
      "of translation, ribosomal\n",
      "large subunit assembly, RNA\n",
      "catabolic process, cellular\n",
      "response to cold                               & nan                     & nan             \\\\\n",
      "  85 & appC    & cytochrome \\ensuremath{<}i\\ensuremath{>}bd\\ensuremath{<}/i\\ensuremath{>}-II\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase                                & transmembrane transport,\n",
      "aerobic electron transport\n",
      "chain, oxidative\n",
      "phosphorylation                               & nan                     & AppY DNA-binding-site\n",
      "regulating appCp2, YdeO DNA-\n",
      "binding-site 27.5 bases\n",
      "upstream of appCp2, YdeO DNA-\n",
      "binding-site 27.5 bases\n",
      "upstream of appCp2, AppY DNA-\n",
      "binding-site regulating appCp2                 \\\\\n",
      "  86 & cyoA    & cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}sub\\ensuremath{>}3\\ensuremath{<}/sub\\ensuremath{>}\\ensuremath{<}/i\\ensuremath{>}\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase                                & ATP synthesis coupled electron\n",
      "transport, electron transport\n",
      "chain, aerobic respiration,\n",
      "electron transport coupled\n",
      "proton transport, aerobic\n",
      "electron transport chain                               & proline to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer, NADH to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer II, NADH to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer I,\n",
      "glycerol-3-phosphate to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer, D-lactate\n",
      "to cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>}\n",
      "oxidase electron transfer,\n",
      "pyruvate to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer, succinate to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer                         & Cra DNA-binding-site\n",
      "regulating cyoAp (BS0-10977),\n",
      "HprR-P DNA-binding-site 69.5\n",
      "bases downstream of cyoAp,\n",
      "CusR-Pasp DNA-binding-site\n",
      "69.5 bases downstream of\n",
      "cyoAp, Cra DNA-binding-site\n",
      "268.5 bases upstream of cyoAp,\n",
      "PdhR DNA-binding-site 177\n",
      "bases upstream of cyoAp, Fnr\n",
      "DNA-binding-site 192.5 bases\n",
      "upstream of cyoAp, ArcA-Pasp54\n",
      "DNA-binding-site 200 bases\n",
      "upstream of cyoAp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 213 bases\n",
      "upstream of cyoAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 191.5 bases\n",
      "upstream of cyoAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 37.5 bases\n",
      "upstream of cyoAp                 \\\\\n",
      "  87 & metA    & homoserine\n",
      "O-succinyltransferase                                & L-methionine biosynthetic\n",
      "process from homoserine via\n",
      "O-succinyl-L-homoserine and\n",
      "cystathionine, methionine\n",
      "biosynthetic process                               & aspartate superpathway,\n",
      "superpathway of L-lysine,\n",
      "L-threonine and L-methionine\n",
      "biosynthesis I, superpathway\n",
      "of L-homoserine and\n",
      "L-methionine biosynthesis,\n",
      "superpathway of\n",
      "\\ensuremath{<}i\\ensuremath{>}S\\ensuremath{<}/i\\ensuremath{>}-adenosyl-L-methionine\n",
      "biosynthesis, L-methionine\n",
      "biosynthesis I                         & MetJ-SAM DNA-binding-site 20.5\n",
      "bases downstream of metAp1,\n",
      "MetJ-SAM DNA-binding-site 4.5\n",
      "bases downstream of metAp1                 \\\\\n",
      "  88 & ymdF    & stress-induced bacterial\n",
      "acidophilic repeat motifs-\n",
      "containing protein YmdF                                & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation                               & nan                     & nan             \\\\\n",
      "  89 & lolC    & lipoprotein release complex    & protein localization to outer\n",
      "membrane, lipoprotein\n",
      "transport, lipoprotein\n",
      "localization to outer membrane                               & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria)                         & nan             \\\\\n",
      "  90 & rho     & transcription termination\n",
      "factor Rho                                & DNA-templated transcription\n",
      "termination                               & nan                     & nan             \\\\\n",
      "  91 & ssrA    & tmRNA                          & rescue of stalled ribosome,\n",
      "trans-translation-dependent\n",
      "protein tagging, trans-\n",
      "translation                               & nan                     & nan             \\\\\n",
      "  92 & yibJ    & putative RHS domain-containing\n",
      "protein YibJ                                & nan                           & nan                     & nan             \\\\\n",
      "  93 & intA    & CP4-57 prophage; integrase\n",
      "IntA                                & viral genome integration into\n",
      "host DNA, symbiont entry into\n",
      "host cell, establishment of\n",
      "integrated proviral latency,\n",
      "DNA integration, DNA\n",
      "recombination, provirus\n",
      "excision                               & nan                     & nan             \\\\\n",
      "  94 & cycA    & D-\n",
      "serine/alanine/glycine:H\\ensuremath{<}sup\\ensuremath{>}+\n",
      "\\ensuremath{<}/sup\\ensuremath{>}symporter                                & transmembrane transport, amino\n",
      "acid transport, D-alanine\n",
      "transmembrane transport,\n",
      "L-alanine transport, beta-\n",
      "alanine transport, glycine\n",
      "transport, D-serine\n",
      "transmembrane transport                               & nan                     & nan             \\\\\n",
      "  95 & abrB    & putative regulator             & regulation of gene expression & nan                     & nan             \\\\\n",
      "  96 & zupT    & divalent metal ion transporter\n",
      "ZupT                                & iron ion transport, zinc ion\n",
      "transport, cobalt ion\n",
      "transport, zinc ion import\n",
      "across plasma membrane, iron\n",
      "ion import across plasma\n",
      "membrane, zinc ion\n",
      "transmembrane transport,\n",
      "transmembrane transport, metal\n",
      "ion transport                               & nan                     & nan             \\\\\n",
      "  97 & ansP    & L-asparagine transporter       & transmembrane transport, amino\n",
      "acid transport                               & nan                     & nan             \\\\\n",
      "  98 & wbbK    & putative glycosyltransferase\n",
      "WbbK                                & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      "  99 & wzzB    & O-antigen co-polymerase        & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      " 100 & murD    & UDP-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-acetylmuramoyl-L-\n",
      "alanine\\&mdash;D-glutamate\n",
      "ligase                                & cell wall organization,\n",
      "biosynthetic process, cell\n",
      "division, regulation of cell\n",
      "shape, peptidoglycan\n",
      "biosynthetic process                               & peptidoglycan biosynthesis I\n",
      "(\\ensuremath{<}I\\ensuremath{>}meso\\ensuremath{<}/I\\ensuremath{>}-diaminopimelate\n",
      "containing),\n",
      "UDP-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-acetylmuramoyl-\n",
      "pentapeptide biosynthesis I\n",
      "(\\ensuremath{<}i\\ensuremath{>}meso\\ensuremath{<}/i\\ensuremath{>}-diaminopimelate\n",
      "containing)                         & LexA DNA-binding-site 3 bases\n",
      "downstream of ftsLp2, MraZ\n",
      "DNA-binding-site 18 bases\n",
      "downstream of mraZp, MraZ DNA-\n",
      "binding-site 8 bases\n",
      "downstream of mraZp, MraZ DNA-\n",
      "binding-site 3 bases upstream\n",
      "of mraZp, PdhR DNA-binding-\n",
      "site 42 bases upstream of\n",
      "mraZp                 \\\\\n",
      " 101 & rodZ    & cytoskeleton protein RodZ      & peptidoglycan biosynthetic\n",
      "process, FtsZ-dependent\n",
      "cytokinesis, regulation of\n",
      "cell shape                               & nan                     & nan             \\\\\n",
      " 102 & mutL    & methyl-directed mismatch\n",
      "repair system                                & nucleotide-excision repair,\n",
      "DNA duplex unwinding,\n",
      "regulation of DNA\n",
      "recombination, DNA repair,\n",
      "mismatch repair involved in\n",
      "maintenance of fidelity\n",
      "involved in DNA-dependent DNA\n",
      "replication, mismatch repair                               & nan                     & nan             \\\\\n",
      " 103 & yibX    & protein YibX-S                 & nan                           & nan                     & nan             \\\\\n",
      " 104 & def     & peptide deformylase            & translation, co-translational\n",
      "protein modification,\n",
      "N-terminal protein amino acid\n",
      "modification                               & nan                     & nan             \\\\\n",
      " 105 & lspA    & lipoprotein signal peptidase   & proteolysis, lipoprotein\n",
      "biosynthetic process via\n",
      "signal peptide cleavage                               & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria)                         & nan             \\\\\n",
      " 106 & cbrC    & PF03691 family colicin E2\n",
      "tolerance protein CbrC                                & bacteriocin immunity          & nan                     & nan             \\\\\n",
      " 107 & sgcX    & KpLE2 phage-like element;\n",
      "putative endoglucanase with\n",
      "Zn-dependent exopeptidase\n",
      "domain                                & proteolysis                   & nan                     & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site regulating\n",
      "sgcXBCQAER, SlyA DNA-binding-\n",
      "site regulating sgcXBCQAER\n",
      "(BS0-8641), SlyA DNA-binding-\n",
      "site regulating sgcXBCQAER\n",
      "(BS0-8639)                 \\\\\n",
      " 108 & abpA    & CP4-57 prophage; anti-\n",
      "bacteriophage protein                                & defense response to virus     & nan                     & nan             \\\\\n",
      " 109 & intZ    & CPZ-55 prophage; putative\n",
      "phage integrase IntZ                                & viral genome integration into\n",
      "host DNA, establishment of\n",
      "integrated proviral latency,\n",
      "symbiont entry into host cell,\n",
      "DNA integration, DNA\n",
      "recombination                               & nan                     & nan             \\\\\n",
      " 110 & rbsB    & ribose ABC transporter         & chemotaxis, D-ribose\n",
      "transmembrane transport,\n",
      "positive chemotaxis                               & nan                     & RbsR DNA-binding-site 8.5\n",
      "bases downstream of rbsDp,\n",
      "CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 62.5 bases\n",
      "upstream of rbsDp                 \\\\\n",
      " 111 & rpe     & ribulose-phosphate 3-epimerase & pentose-phosphate shunt,\n",
      "carbohydrate metabolic\n",
      "process, pentose catabolic\n",
      "process, pentose-phosphate\n",
      "shunt, non-oxidative branch                               & pentose phosphate pathway,\n",
      "pentose phosphate pathway\n",
      "(non-oxidative branch) I                         & nan             \\\\\n",
      " 112 & tar     & chemotaxis signaling complex\n",
      "core unit - aspartate sensing                                & positive regulation of post-\n",
      "translational protein\n",
      "modification, cellular\n",
      "response to amino acid\n",
      "stimulus, regulation of\n",
      "chemotaxis, detection of\n",
      "chemical stimulus, signal\n",
      "complex assembly, signal\n",
      "transduction, protein\n",
      "homooligomerization,\n",
      "chemotaxis                               & nan                     & Fnr DNA-binding-site\n",
      "regulating tarp                 \\\\\n",
      " 113 & rnlA    & CP4-57 prophage; RNase LS,\n",
      "toxin of the RnlAB toxin-\n",
      "antitoxin system, RnlA-RnlB\n",
      "toxin-antitoxin complex                                & regulation of DNA-templated\n",
      "transcription, single-species\n",
      "biofilm formation, regulation\n",
      "of growth, defense response to\n",
      "virus, mRNA catabolic process                               & nan                     & IscR DNA-binding-site 29.5\n",
      "bases upstream of rnlAp2                 \\\\\n",
      " 114 & hyfF    & hydrogenase 4                  & anaerobic electron transport\n",
      "chain, formate oxidation,\n",
      "anaerobic respiration, glucose\n",
      "catabolic process, DNA damage\n",
      "response                               & nan                     & Fnr DNA-binding-site 105.5\n",
      "bases upstream of hyfAp, HyfR\n",
      "DNA-binding-site 129.5 bases\n",
      "upstream of hyfAp, FhlA-\n",
      "formate DNA-binding-site 130\n",
      "bases upstream of hyfAp, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 160.5 bases\n",
      "upstream of hyfAp                 \\\\\n",
      " 115 & rcsA    & RcsAB DNA-binding\n",
      "transcriptional dual regulator                                & positive regulation of DNA-\n",
      "templated transcription,\n",
      "negative regulation of DNA-\n",
      "templated transcription,\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility, regulation of\n",
      "capsule organization,\n",
      "regulation of DNA-templated\n",
      "transcription, DNA-templated\n",
      "transcription                               & nan                     & H-NS DNA-binding-site\n",
      "regulating rcsAp, RcsAB DNA-\n",
      "binding-site 125 bases\n",
      "upstream of rcsAp, GadE DNA-\n",
      "binding-site regulating rcsAp,\n",
      "RcsAB DNA-binding-site 172\n",
      "bases upstream of rcsAp                 \\\\\n",
      " 116 & ychO    & putative invasin YchO          & nan                           & nan                     & Fnr DNA-binding-site\n",
      "regulating ychOp                 \\\\\n",
      " 117 & pgpB    & phosphatidylglycerophosphatase\n",
      "B                                & phospholipid catabolic\n",
      "process, phosphatidylglycerol\n",
      "biosynthetic process,\n",
      "peptidoglycan biosynthetic\n",
      "process, glycerophospholipid\n",
      "biosynthetic process                               & superpathway of phospholipid\n",
      "biosynthesis III (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), cardiolipin\n",
      "biosynthesis III, cardiolipin\n",
      "biosynthesis I, polyisoprenoid\n",
      "biosynthesis (\\ensuremath{<}i\\ensuremath{>}E. coli\\ensuremath{<}/i\\ensuremath{>}),\n",
      "di-\\ensuremath{<}i\\ensuremath{>}trans\\ensuremath{<}/i\\ensuremath{>},poly-\n",
      "\\ensuremath{<}i\\ensuremath{>}cis\\ensuremath{<}/i\\ensuremath{>}-undecaprenyl\n",
      "phosphate biosynthesis                         & nan             \\\\\n",
      " 118 & gyrA    & DNA gyrase                     & chromosome organization,\n",
      "response to xenobiotic\n",
      "stimulus, negative regulation\n",
      "of DNA-templated DNA\n",
      "replication, DNA-templated DNA\n",
      "replication, DNA topological\n",
      "change, response to\n",
      "antibiotic, DNA-templated\n",
      "transcription                               & nan                     & Fis DNA-binding-site 42.5\n",
      "bases upstream of gyrAp, CspA\n",
      "DNA-binding-site 108 bases\n",
      "upstream of gyrAp, CspA DNA-\n",
      "binding-site 88 bases upstream\n",
      "of gyrAp, CspA DNA-binding-\n",
      "site 60 bases upstream of\n",
      "gyrAp                 \\\\\n",
      " 119 & alaA    & glutamate\\&mdash;pyruvate\n",
      "aminotransferase AlaA                                & amino acid biosynthetic\n",
      "process, biosynthetic process,\n",
      "alanine biosynthetic process,\n",
      "D-alanine biosynthetic\n",
      "process, DNA damage response,\n",
      "response to antibiotic,\n",
      "L-alanine biosynthetic process\n",
      "from pyruvate                               & superpathway of L-alanine\n",
      "biosynthesis, L-alanine\n",
      "biosynthesis II                         & nan             \\\\\n",
      " 120 & yciQ    & DUF2207 domain-containing\n",
      "protein YciQ                                & nan                           & nan                     & nan             \\\\\n",
      " 121 & ppdC    & prepilin-type N-terminal\n",
      "cleavage/methylation domain-\n",
      "containing protein PpdC                                & nan                           & nan                     & nan             \\\\\n",
      " 122 & ynaE    & Rac prophage; uncharacterized\n",
      "protein YnaE                                & response to cold              & nan                     & GlaR DNA-binding-site\n",
      "regulating ynaE                 \\\\\n",
      " 123 & oppA    & oligopeptide ABC transporter   & oligopeptide import across\n",
      "plasma membrane, transmembrane\n",
      "transport, peptide transport,\n",
      "protein transport, chaperone-\n",
      "mediated protein folding,\n",
      "response to heat, oligopeptide\n",
      "transport                               & nan                     & DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site 136 bases\n",
      "upstream of oppAp, Lrp DNA-\n",
      "binding-site 1818 bases\n",
      "upstream of oppAp, ArcA-Pasp54\n",
      "DNA-binding-site regulating\n",
      "oppAp                 \\\\\n",
      " 124 & acnB    & bifunctional aconitate\n",
      "hydratase B/2-methylisocitrate\n",
      "dehydratase                                & glyoxylate cycle,\n",
      "tricarboxylic acid cycle,\n",
      "regulation of translation,\n",
      "propionate catabolic process,\n",
      "2-methylcitrate cycle                               & superpathway of glycolysis,\n",
      "pyruvate dehydrogenase, TCA,\n",
      "and glyoxylate bypass,\n",
      "superpathway of glyoxylate\n",
      "bypass and TCA, glyoxylate\n",
      "cycle, TCA cycle I\n",
      "(prokaryotic), mixed acid\n",
      "fermentation, 2-methylcitrate\n",
      "cycle I                         & ArcA-Pasp54 DNA-binding-site\n",
      "48 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "57 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "119 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "113 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "139 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "143 bases upstream of acnBp1,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "170 bases upstream of acnBp1,\n",
      "Fis DNA-binding-site 29.5\n",
      "bases upstream of acnBp1, Fis\n",
      "DNA-binding-site 73.5 bases\n",
      "upstream of acnBp1, Fis DNA-\n",
      "binding-site 171.5 bases\n",
      "upstream of acnBp1, Cra DNA-\n",
      "binding-site regulating\n",
      "acnBp1, CRP-cyclic-AMP DNA-\n",
      "binding transcriptional dual\n",
      "regulator DNA-binding-site\n",
      "37.5 bases upstream of acnBp1                 \\\\\n",
      " 125 & gltT    & tRNA-Glu(UUC)                  & nan                           & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging                         & Fis DNA-binding-site 143 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 53 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "H-NS DNA-binding-site 50 bases\n",
      "upstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 75 bases downstream of\n",
      "rrsBp, Lrp-leu DNA-binding-\n",
      "site 269 bases downstream of\n",
      "rrsBp, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsBp2, NhaR-\n",
      "Na+ DNA-binding-site\n",
      "regulating rrsBp1, Lrp-leu\n",
      "DNA-binding-site 269 bases\n",
      "downstream of rrsBp, Lrp-leu\n",
      "DNA-binding-site 75 bases\n",
      "downstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, H-NS DNA-binding-\n",
      "site 50 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsBp, Fis\n",
      "DNA-binding-site 71 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsBp                 \\\\\n",
      " 126 & ffh     & Signal Recognition Particle    & SRP-dependent cotranslational\n",
      "protein targeting to membrane,\n",
      "protein targeting to membrane                               & nan                     & nan             \\\\\n",
      " 127 & mntH    & Mn(2+):H(+) symporter          & iron ion transmembrane\n",
      "transport, proton\n",
      "transmembrane transport,\n",
      "cadmium ion transmembrane\n",
      "transport, manganese ion\n",
      "transport, iron ion transport,\n",
      "cobalt ion transport, cadmium\n",
      "ion transport, monoatomic ion\n",
      "transport, metal ion\n",
      "transport, manganese import\n",
      "into cell, cellular response\n",
      "to oxidative stress                               & nan                     & OxyR DNA-binding-site 76 bases\n",
      "upstream of mntHp, OxyR DNA-\n",
      "binding-site 54 bases upstream\n",
      "of mntHp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 31.5\n",
      "bases upstream of mntHp, MntR\n",
      "DNA-binding-site 5.5 bases\n",
      "downstream of mntHp                 \\\\\n",
      " 128 & ydhQ    & putative adhesin-related\n",
      "protein YdhQ                                & DNA damage response           & nan                     & nan             \\\\\n",
      " 129 & fliC    & flagellum                      & nan                           & nan                     & MatA DNA-binding-site\n",
      "regulating fliCp, GadE DNA-\n",
      "binding-site regulating fliCp                 \\\\\n",
      " 130 & rrlE    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 170 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 117 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 101 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 72 bases upstream\n",
      "of rrsEp3, NhaR-Na+ DNA-\n",
      "binding-site regulating\n",
      "rrsEp2, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsEp1, Fis\n",
      "DNA-binding-site 72 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 101 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 117 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsEp3, Fis DNA-\n",
      "binding-site 170 bases\n",
      "upstream of rrsEp3                 \\\\\n",
      " 131 & nadK    & NAD kinase                     & NAD metabolic process, NADP\n",
      "biosynthetic process                               & NAD phosphorylation and\n",
      "dephosphorylation                         & nan             \\\\\n",
      " 132 & bax     & putative glycoside hydrolase\n",
      "Bax                                & nan                           & nan                     & nan             \\\\\n",
      " 133 & rfbC    & dTDP-4-dehydrorhamnose\n",
      "3,5-epimerase                                & dTDP-rhamnose biosynthetic\n",
      "process, extracellular\n",
      "polysaccharide biosynthetic\n",
      "process, lipopolysaccharide\n",
      "biosynthetic process, O\n",
      "antigen biosynthetic process,\n",
      "response to UV, response to\n",
      "antibiotic, DNA damage\n",
      "response                               & \\ensuremath{<}i\\ensuremath{>}O\\ensuremath{<}/i\\ensuremath{>}-antigen building\n",
      "blocks biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), dTDP-L-rhamnose\n",
      "biosynthesis                         & nan             \\\\\n",
      " 134 & radA    & DNA recombination protein      & response to radiation, DNA\n",
      "repair, recombinational repair                               & nan                     & nan             \\\\\n",
      " 135 & glf     & UDP-galactopyranose mutase     & lipopolysaccharide\n",
      "biosynthetic process                               & \\ensuremath{<}i\\ensuremath{>}O\\ensuremath{<}/i\\ensuremath{>}-antigen building\n",
      "blocks biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>})                         & nan             \\\\\n",
      " 136 & trpD    & anthranilate synthase subunit\n",
      "TrpE, anthranilate synthase\n",
      "subunit TrpD                                & phenazine biosynthetic\n",
      "process, glutamine metabolic\n",
      "process, tryptophan\n",
      "biosynthetic process                               & L-glutamine degradation I,\n",
      "L-glutamate biosynthesis I,\n",
      "ammonia assimilation cycle\n",
      "III, superpathway of\n",
      "chorismate metabolism,\n",
      "superpathway of aromatic amino\n",
      "acid biosynthesis,\n",
      "superpathway of L-tryptophan\n",
      "biosynthesis, L-tryptophan\n",
      "biosynthesis                         & TrpR-trp DNA-binding-site 11.5\n",
      "bases upstream of trpLp, TrpR-\n",
      "trp DNA-binding-site 3.5 bases\n",
      "upstream of trpLp, TrpR-trp\n",
      "DNA-binding-site 19.5 bases\n",
      "upstream of trpLp                 \\\\\n",
      " 137 & rrsB    & 16S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 143 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 53 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "H-NS DNA-binding-site 50 bases\n",
      "upstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, Lrp-leu DNA-binding-\n",
      "site 75 bases downstream of\n",
      "rrsBp, Lrp-leu DNA-binding-\n",
      "site 269 bases downstream of\n",
      "rrsBp, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsBp2, NhaR-\n",
      "Na+ DNA-binding-site\n",
      "regulating rrsBp1, Lrp-leu\n",
      "DNA-binding-site 269 bases\n",
      "downstream of rrsBp, Lrp-leu\n",
      "DNA-binding-site 75 bases\n",
      "downstream of rrsBp, H-NS DNA-\n",
      "binding-site 83 bases upstream\n",
      "of rrsBp, H-NS DNA-binding-\n",
      "site 50 bases upstream of\n",
      "rrsBp, H-NS DNA-binding-site\n",
      "26 bases upstream of rrsBp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsBp, Fis\n",
      "DNA-binding-site 71 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 102 bases\n",
      "upstream of rrsBp, Fis DNA-\n",
      "binding-site 143 bases\n",
      "upstream of rrsBp                 \\\\\n",
      " 138 & rtcR    & DNA-binding transcriptional\n",
      "activator RtcR                                & regulation of DNA-templated\n",
      "transcription, phosphorelay\n",
      "signal transduction system,\n",
      "positive regulation of DNA-\n",
      "templated transcription                               & nan                     & nan             \\\\\n",
      " 139 & gabP    & 4-aminobutanoate:H\\ensuremath{<}sup\\ensuremath{>}+\\ensuremath{<}/sup\\ensuremath{>}\n",
      "symporter                                & transmembrane transport,\n",
      "gamma-aminobutyric acid\n",
      "catabolic process, DNA damage\n",
      "response, amino acid\n",
      "transport, cellular response\n",
      "to nitrogen starvation,\n",
      "response to carbon starvation,\n",
      "gamma-aminobutyric acid\n",
      "transport                               & nan                     & Nac DNA-binding-site 44 bases\n",
      "upstream of gabDp, GlaR DNA-\n",
      "binding-site 32 bases upstream\n",
      "of csiDp, GlaR DNA-binding-\n",
      "site 13 bases upstream of\n",
      "csiDp, CRP-cyclic-AMP DNA-\n",
      "binding transcriptional dual\n",
      "regulator DNA-binding-site\n",
      "68.5 bases upstream of csiDp,\n",
      "Lrp-leu DNA-binding-site\n",
      "regulating csiDp, H-NS DNA-\n",
      "binding-site regulating csiDp,\n",
      "Nac DNA-binding-site 44 bases\n",
      "upstream of gabDp                 \\\\\n",
      " 140 & entF    & seryl-[EntF peptidyl-carrier\n",
      "protein]                                & secondary metabolite\n",
      "biosynthetic process,\n",
      "biosynthetic process, amino\n",
      "acid activation for\n",
      "nonribosomal peptide\n",
      "biosynthetic process,\n",
      "enterobactin biosynthetic\n",
      "process                               & superpathway of chorismate\n",
      "metabolism, acyl carrier\n",
      "protein metabolism,\n",
      "enterobactin biosynthesis                         & H-NS DNA-binding-site 0.5\n",
      "bases upstream of fesp, DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "28 bases upstream of fesp,\n",
      "DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site 9 bases upstream\n",
      "of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 15 bases\n",
      "upstream of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 2 bases\n",
      "upstream of fesp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 21 bases\n",
      "upstream of fesp, Fnr DNA-\n",
      "binding-site 143.5 bases\n",
      "upstream of fesp                 \\\\\n",
      " 141 & yqiI    & putative fimbrial protein YqiI & cell adhesion, response to\n",
      "methylglyoxal                               & nan                     & nan             \\\\\n",
      " 142 & idi     & isopentenyl-diphosphate\n",
      "\\&Delta;-isomerase                                & dimethylallyl diphosphate\n",
      "biosynthetic process, DNA\n",
      "damage response, isoprenoid\n",
      "biosynthetic process                               & methylerythritol phosphate\n",
      "pathway I                         & nan             \\\\\n",
      " 143 & gltW    & tRNA-Glu(UUC)                  & nan                           & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging                         & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      " 144 & rrlG    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      " 145 & dgcQ    & diguanylate cyclase DgcQ       & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation, cellulose\n",
      "biosynthetic process, negative\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility                               & nan                     & nan             \\\\\n",
      " 146 & rrsA    & ribosome                       & nan                           & nan                     & Fis DNA-binding-site 133 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 111 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsAp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsAp,\n",
      "Lrp-leu DNA-binding-site 53\n",
      "bases upstream of rrsAp, Lrp-\n",
      "leu DNA-binding-site 73 bases\n",
      "upstream of rrsAp, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsAp2, NhaR-Na+ DNA-binding-\n",
      "site regulating rrsAp1, Lrp-\n",
      "leu DNA-binding-site 73 bases\n",
      "upstream of rrsAp, Lrp-leu\n",
      "DNA-binding-site 53 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 71 bases upstream\n",
      "of rrsAp, Fis DNA-binding-site\n",
      "91 bases upstream of rrsAp,\n",
      "Fis DNA-binding-site 111 bases\n",
      "upstream of rrsAp, Fis DNA-\n",
      "binding-site 133 bases\n",
      "upstream of rrsAp                 \\\\\n",
      " 147 & ynjF    & CDP-alcohol\n",
      "phosphatidyltransferase\n",
      "domain-containing protein YnjF                                & phospholipid biosynthetic\n",
      "process                               & nan                     & nan             \\\\\n",
      " 148 & aldA    & aldehyde dehydrogenase A, NAD-\n",
      "linked                                & gamma-aminobutyric acid\n",
      "catabolic process, fucose\n",
      "catabolic process, rhamnose\n",
      "catabolic process, L-fucose\n",
      "catabolic process                               & superpathway of fucose and\n",
      "rhamnose degradation,\n",
      "methylglyoxal degradation IV,\n",
      "superpathway of methylglyoxal\n",
      "degradation, L-lactaldehyde\n",
      "degradation (aerobic),\n",
      "superpathway of glycol\n",
      "metabolism and degradation,\n",
      "ethylene glycol degradation,\n",
      "D-arabinose degradation II                         & SoxS DNA-binding-site 161\n",
      "bases upstream of aldAp, Rob\n",
      "DNA-binding-site 161 bases\n",
      "upstream of aldAp, MarA DNA-\n",
      "binding-site 161 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 157 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 5 bases\n",
      "upstream of aldAp, DnaA-ATP\n",
      "DNA-binding-site 7.5 bases\n",
      "upstream of aldAp, ArcA-Pasp54\n",
      "DNA-binding-site 2 bases\n",
      "downstream of aldAp, ArcA-\n",
      "Pasp54 DNA-binding-site 13\n",
      "bases downstream of aldAp,\n",
      "ArcA-Pasp54 DNA-binding-site\n",
      "137 bases upstream of aldAp,\n",
      "Fnr DNA-binding-site 10.5\n",
      "bases upstream of aldAp, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 59.5 bases\n",
      "upstream of aldAp                 \\\\\n",
      " 149 & rrlD    & 23S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 174 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 144 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 126 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsDp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsDp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsDp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsDp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsDp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 126 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 144 bases\n",
      "upstream of rrsDp, Fis DNA-\n",
      "binding-site 174 bases\n",
      "upstream of rrsDp                 \\\\\n",
      " 150 & dnaK    & chaperone protein DnaK         & protein refolding, protein-\n",
      "containing complex assembly,\n",
      "negative regulation of DNA-\n",
      "templated transcription, DNA\n",
      "replication, stress response\n",
      "to copper ion, protein\n",
      "folding, chaperone cofactor-\n",
      "dependent protein refolding,\n",
      "cellular response to unfolded\n",
      "protein, protein unfolding,\n",
      "response to heat                               & nan                     & nan             \\\\\n",
      " 151 & yohC    & putative inner membrane\n",
      "protein                                & DNA damage response           & nan                     & nan             \\\\\n",
      " 152 & nanY    & KpLE2 phage-like element; 2,7-\n",
      "anhydro-\\ensuremath{<}i\\ensuremath{>}N\\ensuremath{<}/i\\ensuremath{>}-\n",
      "acetylneuraminate hydratase                                & DNA damage response,\n",
      "N-acetylneuraminate catabolic\n",
      "process                               & nan                     & NanR DNA-binding-site 376.5\n",
      "bases upstream of yjhBp2, NanR\n",
      "DNA-binding-site 384.5 bases\n",
      "upstream of yjhBp2, NanR DNA-\n",
      "binding-site 393.5 bases\n",
      "upstream of yjhBp2                 \\\\\n",
      " 153 & ynfD    & DUF1161 domain-containing\n",
      "protein YnfD                                & nan                           & nan                     & nan             \\\\\n",
      " 154 & wbbH    & putative O-antigen polymerase  & lipopolysaccharide\n",
      "biosynthetic process                               & nan                     & nan             \\\\\n",
      " 155 & puuB    & \\&gamma;-glutamylputrescine\n",
      "oxidase                                & putrescine catabolic process  & superpathway of L-arginine and\n",
      "L-ornithine degradation,\n",
      "superpathway of ornithine\n",
      "degradation, superpathway of\n",
      "L-arginine, putrescine, and\n",
      "4-aminobutanoate degradation,\n",
      "putrescine degradation II                         & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 63.5 bases\n",
      "downstream of puuDp, PuuR DNA-\n",
      "binding-site 161.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 126.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 84.5 bases\n",
      "upstream of puuAp, PuuR DNA-\n",
      "binding-site 11.5 bases\n",
      "downstream of puuAp, ArcA-\n",
      "Pasp54 DNA-binding-site 82\n",
      "bases downstream of puuDp,\n",
      "PtrR-gln DNA-binding-site\n",
      "regulating puuCp                 \\\\\n",
      " 156 & fhuA    & ferrichrome outer membrane\n",
      "transport complex                                & iron ion transport,\n",
      "intracellular iron ion\n",
      "homeostasis, siderophore-\n",
      "dependent iron import into\n",
      "cell, siderophore transport,\n",
      "siderophore transmembrane\n",
      "transport                               & nan                     & DNA-binding transcriptional\n",
      "dual regulator Fur DNA-\n",
      "binding-site regulating\n",
      "fhuACDB (BS0-10960), DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "regulating fhuACDB\n",
      "(BS0-10871), DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 3 bases\n",
      "upstream of fhuAp                 \\\\\n",
      " 157 & zapA    & cell division protein ZapA     & cell septum assembly, septin\n",
      "ring assembly, division septum\n",
      "assembly, FtsZ-dependent\n",
      "cytokinesis, cell division                               & nan                     & nan             \\\\\n",
      " 158 & rsmJ    & 16S rRNA m\\ensuremath{<}sup\\ensuremath{>}2\\ensuremath{<}/sup\\ensuremath{>}G1516\n",
      "methyltransferase                                & methylation, rRNA processing,\n",
      "rRNA methylation, rRNA base\n",
      "methylation                               & nan                     & nan             \\\\\n",
      " 159 & efeU    & inactive ferrous iron permease\n",
      "EfeU                                & iron ion transmembrane\n",
      "transport                               & nan                     & CpxR-phosphorylated DNA-\n",
      "binding-site 54.5 bases\n",
      "upstream of efeUp                 \\\\\n",
      " 160 & narQ    & sensor histidine kinase NarQ   & phosphorelay signal\n",
      "transduction system, signal\n",
      "transduction, nitrate\n",
      "assimilation, cellular\n",
      "response to nitrate, cellular\n",
      "response to nitrite                               & NarQ Two-Component Signal\n",
      "Transduction System, nitrate\n",
      "dependent                         & CRP-cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 0.5 bases\n",
      "upstream of narQp                 \\\\\n",
      " 161 & rrsG    & 16S ribosomal RNA              & translation                   & nan                     & Fis DNA-binding-site 112 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 91 bases upstream\n",
      "of rrsGp, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "NhaR-Na+ DNA-binding-site\n",
      "regulating rrsGp2, NhaR-Na+\n",
      "DNA-binding-site regulating\n",
      "rrsGp1, Fis DNA-binding-site\n",
      "71 bases upstream of rrsGp,\n",
      "Fis DNA-binding-site 91 bases\n",
      "upstream of rrsGp, Fis DNA-\n",
      "binding-site 112 bases\n",
      "upstream of rrsGp                 \\\\\n",
      " 162 & fliD    & flagellum                      & cell adhesion, bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility                               & nan                     & FlhDC DNA-binding-site 55\n",
      "bases upstream of fliDp2                 \\\\\n",
      " 163 & purR    & DNA-binding transcriptional\n",
      "repressor PurR-hypoxanthine                                & regulation of DNA-templated\n",
      "transcription, purine\n",
      "nucleotide biosynthetic\n",
      "process, negative regulation\n",
      "of purine nucleotide\n",
      "biosynthetic process, negative\n",
      "regulation of DNA-templated\n",
      "transcription                               & nan                     & PurR-hypoxanthine DNA-binding-\n",
      "site 191.5 bases downstream of\n",
      "purRp, PurR-hypoxanthine DNA-\n",
      "binding-site 103.5 bases\n",
      "downstream of purRp, DNA-\n",
      "binding transcriptional dual\n",
      "regulator Fur DNA-binding-site\n",
      "49 bases upstream of purRp                 \\\\\n",
      " 164 & appY    & DLP12 prophage; DNA-binding\n",
      "transcriptional activator AppY                                & regulation of DNA-templated\n",
      "transcription, response to\n",
      "X-ray, cellular response to\n",
      "phosphate starvation, positive\n",
      "regulation of DNA-templated\n",
      "transcription                               & nan                     & ArcA-Pasp54 DNA-binding-site\n",
      "197 bases upstream of appYp2,\n",
      "DpiA-Pasp DNA-binding-site\n",
      "regulating appYp1, H-NS DNA-\n",
      "binding-site regulating appYp1                 \\\\\n",
      " 165 & ftnB    & putative ferritin-like protein & intracellular sequestering of\n",
      "iron ion                               & nan                     & CpxR-phosphorylated DNA-\n",
      "binding-site 61 bases upstream\n",
      "of ftnBp2                 \\\\\n",
      " 166 & folX    & dihydroneopterin triphosphate\n",
      "2'-epimerase                                & pteridine-containing compound\n",
      "biosynthetic process, folic\n",
      "acid-containing compound\n",
      "metabolic process                               & tetrahydromonapterin\n",
      "biosynthesis                         & nan             \\\\\n",
      " 167 & argH    & argininosuccinate lyase        & arginine biosynthetic process\n",
      "via ornithine, arginine\n",
      "biosynthetic process                               & superpathway of arginine and\n",
      "polyamine biosynthesis,\n",
      "L-arginine biosynthesis I (via\n",
      "L-ornithine)                         & ArgR-arg DNA-binding-site 6.5\n",
      "bases upstream of argEp1,\n",
      "ArgR-arg DNA-binding-site 16.5\n",
      "bases downstream of argEp1                 \\\\\n",
      " 168 & nrdA    & ribonucleoside-diphosphate\n",
      "reductase 1                                & protein folding,\n",
      "2'-deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "ribonucleoside diphosphate\n",
      "metabolic process,\n",
      "deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "nucleobase-containing small\n",
      "molecule interconversion                               & superpathway of pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis (\\ensuremath{<}i\\ensuremath{>}E.\n",
      "coli\\ensuremath{<}/i\\ensuremath{>}), superpathway of\n",
      "pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis,\n",
      "pyrimidine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis I,\n",
      "superpathway of adenosine\n",
      "nucleotides \\ensuremath{<}i\\ensuremath{>}de novo\\ensuremath{<}/i\\ensuremath{>}\n",
      "biosynthesis II, adenosine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II,\n",
      "superpathway of histidine,\n",
      "purine, and pyrimidine\n",
      "biosynthesis, superpathway of\n",
      "purine nucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II,\n",
      "superpathway of guanosine\n",
      "nucleotides \\ensuremath{<}i\\ensuremath{>}de novo\\ensuremath{<}/i\\ensuremath{>}\n",
      "biosynthesis II, guanosine\n",
      "deoxyribonucleotides \\ensuremath{<}i\\ensuremath{>}de\n",
      "novo\\ensuremath{<}/i\\ensuremath{>} biosynthesis II                         & H-NS DNA-binding-site 415\n",
      "bases upstream of nrdAp, H-NS\n",
      "DNA-binding-site 439 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 334.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 281.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 244.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 14 bases\n",
      "upstream of nrdAp, NrdR\n",
      "transcriptional repressor DNA-\n",
      "binding-site 25.5 bases\n",
      "downstream of nrdAp, NrdR\n",
      "transcriptional repressor DNA-\n",
      "binding-site 7.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 36.5 bases\n",
      "upstream of nrdAp, DnaA-ATP\n",
      "DNA-binding-site 48 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 156 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 163.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 190.5 bases\n",
      "upstream of nrdAp, ArgP DNA-\n",
      "binding-site 218.5 bases\n",
      "upstream of nrdAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 136.5 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 128 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 199 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 269 bases\n",
      "upstream of nrdAp, Fis DNA-\n",
      "binding-site 323 bases\n",
      "upstream of nrdAp                 \\\\\n",
      " 169 & mdtF    & multidrug efflux pump MdtEF-\n",
      "TolC                                & xenobiotic transmembrane\n",
      "transport, transmembrane\n",
      "transport, xenobiotic\n",
      "transport, response to toxic\n",
      "substance, bile acid and bile\n",
      "salt transport, response to\n",
      "antibiotic                               & nan                     & FliZ DNA-binding-site 9.5\n",
      "bases downstream of gadEp2,\n",
      "GadW DNA-binding-site 574.5\n",
      "bases upstream of gadEp2, GadX\n",
      "DNA-binding-site 29.5 bases\n",
      "upstream of gadEp3, H-NS DNA-\n",
      "binding-site regulating\n",
      "gadEp2, GadX DNA-binding-site\n",
      "50.5 bases upstream of gadEp3,\n",
      "GadW DNA-binding-site 595.5\n",
      "bases upstream of gadEp2, CRP-\n",
      "cyclic-AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 19.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 30.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 63.5 bases\n",
      "upstream of gadEp2, GadE DNA-\n",
      "binding-site 198.5 bases\n",
      "upstream of gadEp2, EvgA-Pasp\n",
      "DNA-binding-site regulating\n",
      "gadEp2, YdeO DNA-binding-site\n",
      "regulating gadEp2                 \\\\\n",
      " 170 & gpr     & L-glyceraldehyde 3-phosphate\n",
      "reductase                                & DNA damage response,\n",
      "methylglyoxal catabolic\n",
      "process                               & superpathway of methylglyoxal\n",
      "degradation, methylglyoxal\n",
      "degradation III                         & nan             \\\\\n",
      " 171 & ycjG    & L-Ala-D/L-Glu epimerase        & cell wall organization, amino\n",
      "acid catabolic process,\n",
      "peptidoglycan turnover                               & muropeptide degradation & PgrR DNA-binding-site\n",
      "regulating ycjGp                 \\\\\n",
      " 172 & deaD    & ATP-dependent RNA helicase\n",
      "DeaD                                & response to cold, mRNA\n",
      "stabilization, response to\n",
      "radiation, positive regulation\n",
      "of translation, ribosomal\n",
      "large subunit assembly, RNA\n",
      "catabolic process, cellular\n",
      "response to cold                               & nan                     & nan             \\\\\n",
      " 173 & appC    & cytochrome \\ensuremath{<}i\\ensuremath{>}bd\\ensuremath{<}/i\\ensuremath{>}-II\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase                                & transmembrane transport,\n",
      "aerobic electron transport\n",
      "chain, oxidative\n",
      "phosphorylation                               & nan                     & AppY DNA-binding-site\n",
      "regulating appCp2, YdeO DNA-\n",
      "binding-site 27.5 bases\n",
      "upstream of appCp2, YdeO DNA-\n",
      "binding-site 27.5 bases\n",
      "upstream of appCp2, AppY DNA-\n",
      "binding-site regulating appCp2                 \\\\\n",
      " 174 & cyoA    & cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}sub\\ensuremath{>}3\\ensuremath{<}/sub\\ensuremath{>}\\ensuremath{<}/i\\ensuremath{>}\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase                                & ATP synthesis coupled electron\n",
      "transport, electron transport\n",
      "chain, aerobic respiration,\n",
      "electron transport coupled\n",
      "proton transport, aerobic\n",
      "electron transport chain                               & proline to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer, NADH to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer II, NADH to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer I,\n",
      "glycerol-3-phosphate to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer, D-lactate\n",
      "to cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>}\n",
      "oxidase electron transfer,\n",
      "pyruvate to cytochrome\n",
      "\\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase electron\n",
      "transfer, succinate to\n",
      "cytochrome \\ensuremath{<}i\\ensuremath{>}bo\\ensuremath{<}/i\\ensuremath{>} oxidase\n",
      "electron transfer                         & Cra DNA-binding-site\n",
      "regulating cyoAp (BS0-10977),\n",
      "HprR-P DNA-binding-site 69.5\n",
      "bases downstream of cyoAp,\n",
      "CusR-Pasp DNA-binding-site\n",
      "69.5 bases downstream of\n",
      "cyoAp, Cra DNA-binding-site\n",
      "268.5 bases upstream of cyoAp,\n",
      "PdhR DNA-binding-site 177\n",
      "bases upstream of cyoAp, Fnr\n",
      "DNA-binding-site 192.5 bases\n",
      "upstream of cyoAp, ArcA-Pasp54\n",
      "DNA-binding-site 200 bases\n",
      "upstream of cyoAp, DNA-binding\n",
      "transcriptional dual regulator\n",
      "Fur DNA-binding-site 213 bases\n",
      "upstream of cyoAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 191.5 bases\n",
      "upstream of cyoAp, CRP-cyclic-\n",
      "AMP DNA-binding\n",
      "transcriptional dual regulator\n",
      "DNA-binding-site 37.5 bases\n",
      "upstream of cyoAp                 \\\\\n",
      " 175 & metA    & homoserine\n",
      "O-succinyltransferase                                & L-methionine biosynthetic\n",
      "process from homoserine via\n",
      "O-succinyl-L-homoserine and\n",
      "cystathionine, methionine\n",
      "biosynthetic process                               & aspartate superpathway,\n",
      "superpathway of L-lysine,\n",
      "L-threonine and L-methionine\n",
      "biosynthesis I, superpathway\n",
      "of L-homoserine and\n",
      "L-methionine biosynthesis,\n",
      "superpathway of\n",
      "\\ensuremath{<}i\\ensuremath{>}S\\ensuremath{<}/i\\ensuremath{>}-adenosyl-L-methionine\n",
      "biosynthesis, L-methionine\n",
      "biosynthesis I                         & MetJ-SAM DNA-binding-site 20.5\n",
      "bases downstream of metAp1,\n",
      "MetJ-SAM DNA-binding-site 4.5\n",
      "bases downstream of metAp1                 \\\\\n",
      "\\hline\n",
      "\\end{tabular}\n"
     ]
    }
   ],
   "source": [
    "latex_table = tabulate(Data, headers='keys', tablefmt='latex', showindex=True)\n",
    "print(latex_table)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "Data.drop('Binding Sites', axis=1, inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\\begingroup\n",
      "\n",
      "\\fontsize{12.0pt}{14.4pt}\\selectfont\n",
      "\n",
      "\\begin{longtable}{llll}\n",
      "\n",
      "\\toprule\n",
      "Genes & Products & Biological Process & Pathways \\\\ \n",
      "\\midrule\\addlinespace[2.5pt]\n",
      "ymdF & stress-induced bacterial\n",
      "acidophilic repeat motifs-\n",
      "containing protein YmdF & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation & nan \\\\\n",
      "lolC & lipoprotein release complex & protein localization to outer\n",
      "membrane, lipoprotein\n",
      "transport, lipoprotein\n",
      "localization to outer membrane & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria) \\\\\n",
      "rho & transcription termination\n",
      "factor Rho & DNA-templated transcription\n",
      "termination & nan \\\\\n",
      "ssrA & tmRNA & rescue of stalled ribosome,\n",
      "trans-translation-dependent\n",
      "protein tagging, trans-\n",
      "translation & nan \\\\\n",
      "yibJ & putative RHS domain-containing\n",
      "protein YibJ & nan & nan \\\\\n",
      "intA & CP4-57 prophage; integrase\n",
      "IntA & viral genome integration into\n",
      "host DNA, symbiont entry into\n",
      "host cell, establishment of\n",
      "integrated proviral latency,\n",
      "DNA integration, DNA\n",
      "recombination, provirus\n",
      "excision & nan \\\\\n",
      "cycA & D-\n",
      "serine/alanine/glycine:H<sup>+\n",
      "</sup>symporter & transmembrane transport, amino\n",
      "acid transport, D-alanine\n",
      "transmembrane transport,\n",
      "L-alanine transport, beta-\n",
      "alanine transport, glycine\n",
      "transport, D-serine\n",
      "transmembrane transport & nan \\\\\n",
      "abrB & putative regulator & regulation of gene expression & nan \\\\\n",
      "zupT & divalent metal ion transporter\n",
      "ZupT & iron ion transport, zinc ion\n",
      "transport, cobalt ion\n",
      "transport, zinc ion import\n",
      "across plasma membrane, iron\n",
      "ion import across plasma\n",
      "membrane, zinc ion\n",
      "transmembrane transport,\n",
      "transmembrane transport, metal\n",
      "ion transport & nan \\\\\n",
      "ansP & L-asparagine transporter & transmembrane transport, amino\n",
      "acid transport & nan \\\\\n",
      "wbbK & putative glycosyltransferase\n",
      "WbbK & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "wzzB & O-antigen co-polymerase & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "murD & UDP-<i>N</i>-acetylmuramoyl-L-\n",
      "alanine\\&mdash;D-glutamate\n",
      "ligase & cell wall organization,\n",
      "biosynthetic process, cell\n",
      "division, regulation of cell\n",
      "shape, peptidoglycan\n",
      "biosynthetic process & peptidoglycan biosynthesis I\n",
      "(<I>meso</I>-diaminopimelate\n",
      "containing),\n",
      "UDP-<i>N</i>-acetylmuramoyl-\n",
      "pentapeptide biosynthesis I\n",
      "(<i>meso</i>-diaminopimelate\n",
      "containing) \\\\\n",
      "rodZ & cytoskeleton protein RodZ & peptidoglycan biosynthetic\n",
      "process, FtsZ-dependent\n",
      "cytokinesis, regulation of\n",
      "cell shape & nan \\\\\n",
      "mutL & methyl-directed mismatch\n",
      "repair system & nucleotide-excision repair,\n",
      "DNA duplex unwinding,\n",
      "regulation of DNA\n",
      "recombination, DNA repair,\n",
      "mismatch repair involved in\n",
      "maintenance of fidelity\n",
      "involved in DNA-dependent DNA\n",
      "replication, mismatch repair & nan \\\\\n",
      "yibX & protein YibX-S & nan & nan \\\\\n",
      "def & peptide deformylase & translation, co-translational\n",
      "protein modification,\n",
      "N-terminal protein amino acid\n",
      "modification & nan \\\\\n",
      "lspA & lipoprotein signal peptidase & proteolysis, lipoprotein\n",
      "biosynthetic process via\n",
      "signal peptide cleavage & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria) \\\\\n",
      "cbrC & PF03691 family colicin E2\n",
      "tolerance protein CbrC & bacteriocin immunity & nan \\\\\n",
      "sgcX & KpLE2 phage-like element;\n",
      "putative endoglucanase with\n",
      "Zn-dependent exopeptidase\n",
      "domain & proteolysis & nan \\\\\n",
      "abpA & CP4-57 prophage; anti-\n",
      "bacteriophage protein & defense response to virus & nan \\\\\n",
      "intZ & CPZ-55 prophage; putative\n",
      "phage integrase IntZ & viral genome integration into\n",
      "host DNA, establishment of\n",
      "integrated proviral latency,\n",
      "symbiont entry into host cell,\n",
      "DNA integration, DNA\n",
      "recombination & nan \\\\\n",
      "rbsB & ribose ABC transporter & chemotaxis, D-ribose\n",
      "transmembrane transport,\n",
      "positive chemotaxis & nan \\\\\n",
      "rpe & ribulose-phosphate 3-epimerase & pentose-phosphate shunt,\n",
      "carbohydrate metabolic\n",
      "process, pentose catabolic\n",
      "process, pentose-phosphate\n",
      "shunt, non-oxidative branch & pentose phosphate pathway,\n",
      "pentose phosphate pathway\n",
      "(non-oxidative branch) I \\\\\n",
      "tar & chemotaxis signaling complex\n",
      "core unit - aspartate sensing & positive regulation of post-\n",
      "translational protein\n",
      "modification, cellular\n",
      "response to amino acid\n",
      "stimulus, regulation of\n",
      "chemotaxis, detection of\n",
      "chemical stimulus, signal\n",
      "complex assembly, signal\n",
      "transduction, protein\n",
      "homooligomerization,\n",
      "chemotaxis & nan \\\\\n",
      "rnlA & CP4-57 prophage; RNase LS,\n",
      "toxin of the RnlAB toxin-\n",
      "antitoxin system, RnlA-RnlB\n",
      "toxin-antitoxin complex & regulation of DNA-templated\n",
      "transcription, single-species\n",
      "biofilm formation, regulation\n",
      "of growth, defense response to\n",
      "virus, mRNA catabolic process & nan \\\\\n",
      "hyfF & hydrogenase 4 & anaerobic electron transport\n",
      "chain, formate oxidation,\n",
      "anaerobic respiration, glucose\n",
      "catabolic process, DNA damage\n",
      "response & nan \\\\\n",
      "rcsA & RcsAB DNA-binding\n",
      "transcriptional dual regulator & positive regulation of DNA-\n",
      "templated transcription,\n",
      "negative regulation of DNA-\n",
      "templated transcription,\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility, regulation of\n",
      "capsule organization,\n",
      "regulation of DNA-templated\n",
      "transcription, DNA-templated\n",
      "transcription & nan \\\\\n",
      "ychO & putative invasin YchO & nan & nan \\\\\n",
      "pgpB & phosphatidylglycerophosphatase\n",
      "B & phospholipid catabolic\n",
      "process, phosphatidylglycerol\n",
      "biosynthetic process,\n",
      "peptidoglycan biosynthetic\n",
      "process, glycerophospholipid\n",
      "biosynthetic process & superpathway of phospholipid\n",
      "biosynthesis III (<i>E.\n",
      "coli</i>), cardiolipin\n",
      "biosynthesis III, cardiolipin\n",
      "biosynthesis I, polyisoprenoid\n",
      "biosynthesis (<i>E. coli</i>),\n",
      "di-<i>trans</i>,poly-\n",
      "<i>cis</i>-undecaprenyl\n",
      "phosphate biosynthesis \\\\\n",
      "gyrA & DNA gyrase & chromosome organization,\n",
      "response to xenobiotic\n",
      "stimulus, negative regulation\n",
      "of DNA-templated DNA\n",
      "replication, DNA-templated DNA\n",
      "replication, DNA topological\n",
      "change, response to\n",
      "antibiotic, DNA-templated\n",
      "transcription & nan \\\\\n",
      "alaA & glutamate\\&mdash;pyruvate\n",
      "aminotransferase AlaA & amino acid biosynthetic\n",
      "process, biosynthetic process,\n",
      "alanine biosynthetic process,\n",
      "D-alanine biosynthetic\n",
      "process, DNA damage response,\n",
      "response to antibiotic,\n",
      "L-alanine biosynthetic process\n",
      "from pyruvate & superpathway of L-alanine\n",
      "biosynthesis, L-alanine\n",
      "biosynthesis II \\\\\n",
      "yciQ & DUF2207 domain-containing\n",
      "protein YciQ & nan & nan \\\\\n",
      "ppdC & prepilin-type N-terminal\n",
      "cleavage/methylation domain-\n",
      "containing protein PpdC & nan & nan \\\\\n",
      "ynaE & Rac prophage; uncharacterized\n",
      "protein YnaE & response to cold & nan \\\\\n",
      "oppA & oligopeptide ABC transporter & oligopeptide import across\n",
      "plasma membrane, transmembrane\n",
      "transport, peptide transport,\n",
      "protein transport, chaperone-\n",
      "mediated protein folding,\n",
      "response to heat, oligopeptide\n",
      "transport & nan \\\\\n",
      "acnB & bifunctional aconitate\n",
      "hydratase B/2-methylisocitrate\n",
      "dehydratase & glyoxylate cycle,\n",
      "tricarboxylic acid cycle,\n",
      "regulation of translation,\n",
      "propionate catabolic process,\n",
      "2-methylcitrate cycle & superpathway of glycolysis,\n",
      "pyruvate dehydrogenase, TCA,\n",
      "and glyoxylate bypass,\n",
      "superpathway of glyoxylate\n",
      "bypass and TCA, glyoxylate\n",
      "cycle, TCA cycle I\n",
      "(prokaryotic), mixed acid\n",
      "fermentation, 2-methylcitrate\n",
      "cycle I \\\\\n",
      "gltT & tRNA-Glu(UUC) & nan & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging \\\\\n",
      "ffh & Signal Recognition Particle & SRP-dependent cotranslational\n",
      "protein targeting to membrane,\n",
      "protein targeting to membrane & nan \\\\\n",
      "mntH & Mn(2+):H(+) symporter & iron ion transmembrane\n",
      "transport, proton\n",
      "transmembrane transport,\n",
      "cadmium ion transmembrane\n",
      "transport, manganese ion\n",
      "transport, iron ion transport,\n",
      "cobalt ion transport, cadmium\n",
      "ion transport, monoatomic ion\n",
      "transport, metal ion\n",
      "transport, manganese import\n",
      "into cell, cellular response\n",
      "to oxidative stress & nan \\\\\n",
      "ydhQ & putative adhesin-related\n",
      "protein YdhQ & DNA damage response & nan \\\\\n",
      "fliC & flagellum & nan & nan \\\\\n",
      "rrlE & 23S ribosomal RNA & translation & nan \\\\\n",
      "nadK & NAD kinase & NAD metabolic process, NADP\n",
      "biosynthetic process & NAD phosphorylation and\n",
      "dephosphorylation \\\\\n",
      "bax & putative glycoside hydrolase\n",
      "Bax & nan & nan \\\\\n",
      "rfbC & dTDP-4-dehydrorhamnose\n",
      "3,5-epimerase & dTDP-rhamnose biosynthetic\n",
      "process, extracellular\n",
      "polysaccharide biosynthetic\n",
      "process, lipopolysaccharide\n",
      "biosynthetic process, O\n",
      "antigen biosynthetic process,\n",
      "response to UV, response to\n",
      "antibiotic, DNA damage\n",
      "response & <i>O</i>-antigen building\n",
      "blocks biosynthesis (<i>E.\n",
      "coli</i>), dTDP-L-rhamnose\n",
      "biosynthesis \\\\\n",
      "radA & DNA recombination protein & response to radiation, DNA\n",
      "repair, recombinational repair & nan \\\\\n",
      "glf & UDP-galactopyranose mutase & lipopolysaccharide\n",
      "biosynthetic process & <i>O</i>-antigen building\n",
      "blocks biosynthesis (<i>E.\n",
      "coli</i>) \\\\\n",
      "trpD & anthranilate synthase subunit\n",
      "TrpE, anthranilate synthase\n",
      "subunit TrpD & phenazine biosynthetic\n",
      "process, glutamine metabolic\n",
      "process, tryptophan\n",
      "biosynthetic process & L-glutamine degradation I,\n",
      "L-glutamate biosynthesis I,\n",
      "ammonia assimilation cycle\n",
      "III, superpathway of\n",
      "chorismate metabolism,\n",
      "superpathway of aromatic amino\n",
      "acid biosynthesis,\n",
      "superpathway of L-tryptophan\n",
      "biosynthesis, L-tryptophan\n",
      "biosynthesis \\\\\n",
      "rrsB & 16S ribosomal RNA & translation & nan \\\\\n",
      "rtcR & DNA-binding transcriptional\n",
      "activator RtcR & regulation of DNA-templated\n",
      "transcription, phosphorelay\n",
      "signal transduction system,\n",
      "positive regulation of DNA-\n",
      "templated transcription & nan \\\\\n",
      "gabP & 4-aminobutanoate:H<sup>+</sup>\n",
      "symporter & transmembrane transport,\n",
      "gamma-aminobutyric acid\n",
      "catabolic process, DNA damage\n",
      "response, amino acid\n",
      "transport, cellular response\n",
      "to nitrogen starvation,\n",
      "response to carbon starvation,\n",
      "gamma-aminobutyric acid\n",
      "transport & nan \\\\\n",
      "entF & seryl-[EntF peptidyl-carrier\n",
      "protein] & secondary metabolite\n",
      "biosynthetic process,\n",
      "biosynthetic process, amino\n",
      "acid activation for\n",
      "nonribosomal peptide\n",
      "biosynthetic process,\n",
      "enterobactin biosynthetic\n",
      "process & superpathway of chorismate\n",
      "metabolism, acyl carrier\n",
      "protein metabolism,\n",
      "enterobactin biosynthesis \\\\\n",
      "yqiI & putative fimbrial protein YqiI & cell adhesion, response to\n",
      "methylglyoxal & nan \\\\\n",
      "idi & isopentenyl-diphosphate\n",
      "\\&Delta;-isomerase & dimethylallyl diphosphate\n",
      "biosynthetic process, DNA\n",
      "damage response, isoprenoid\n",
      "biosynthetic process & methylerythritol phosphate\n",
      "pathway I \\\\\n",
      "gltW & tRNA-Glu(UUC) & nan & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging \\\\\n",
      "rrlG & 23S ribosomal RNA & translation & nan \\\\\n",
      "dgcQ & diguanylate cyclase DgcQ & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation, cellulose\n",
      "biosynthetic process, negative\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility & nan \\\\\n",
      "rrsA & ribosome & nan & nan \\\\\n",
      "ynjF & CDP-alcohol\n",
      "phosphatidyltransferase\n",
      "domain-containing protein YnjF & phospholipid biosynthetic\n",
      "process & nan \\\\\n",
      "aldA & aldehyde dehydrogenase A, NAD-\n",
      "linked & gamma-aminobutyric acid\n",
      "catabolic process, fucose\n",
      "catabolic process, rhamnose\n",
      "catabolic process, L-fucose\n",
      "catabolic process & superpathway of fucose and\n",
      "rhamnose degradation,\n",
      "methylglyoxal degradation IV,\n",
      "superpathway of methylglyoxal\n",
      "degradation, L-lactaldehyde\n",
      "degradation (aerobic),\n",
      "superpathway of glycol\n",
      "metabolism and degradation,\n",
      "ethylene glycol degradation,\n",
      "D-arabinose degradation II \\\\\n",
      "rrlD & 23S ribosomal RNA & translation & nan \\\\\n",
      "dnaK & chaperone protein DnaK & protein refolding, protein-\n",
      "containing complex assembly,\n",
      "negative regulation of DNA-\n",
      "templated transcription, DNA\n",
      "replication, stress response\n",
      "to copper ion, protein\n",
      "folding, chaperone cofactor-\n",
      "dependent protein refolding,\n",
      "cellular response to unfolded\n",
      "protein, protein unfolding,\n",
      "response to heat & nan \\\\\n",
      "yohC & putative inner membrane\n",
      "protein & DNA damage response & nan \\\\\n",
      "nanY & KpLE2 phage-like element; 2,7-\n",
      "anhydro-<i>N</i>-\n",
      "acetylneuraminate hydratase & DNA damage response,\n",
      "N-acetylneuraminate catabolic\n",
      "process & nan \\\\\n",
      "ynfD & DUF1161 domain-containing\n",
      "protein YnfD & nan & nan \\\\\n",
      "wbbH & putative O-antigen polymerase & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "puuB & \\&gamma;-glutamylputrescine\n",
      "oxidase & putrescine catabolic process & superpathway of L-arginine and\n",
      "L-ornithine degradation,\n",
      "superpathway of ornithine\n",
      "degradation, superpathway of\n",
      "L-arginine, putrescine, and\n",
      "4-aminobutanoate degradation,\n",
      "putrescine degradation II \\\\\n",
      "fhuA & ferrichrome outer membrane\n",
      "transport complex & iron ion transport,\n",
      "intracellular iron ion\n",
      "homeostasis, siderophore-\n",
      "dependent iron import into\n",
      "cell, siderophore transport,\n",
      "siderophore transmembrane\n",
      "transport & nan \\\\\n",
      "zapA & cell division protein ZapA & cell septum assembly, septin\n",
      "ring assembly, division septum\n",
      "assembly, FtsZ-dependent\n",
      "cytokinesis, cell division & nan \\\\\n",
      "rsmJ & 16S rRNA m<sup>2</sup>G1516\n",
      "methyltransferase & methylation, rRNA processing,\n",
      "rRNA methylation, rRNA base\n",
      "methylation & nan \\\\\n",
      "efeU & inactive ferrous iron permease\n",
      "EfeU & iron ion transmembrane\n",
      "transport & nan \\\\\n",
      "narQ & sensor histidine kinase NarQ & phosphorelay signal\n",
      "transduction system, signal\n",
      "transduction, nitrate\n",
      "assimilation, cellular\n",
      "response to nitrate, cellular\n",
      "response to nitrite & NarQ Two-Component Signal\n",
      "Transduction System, nitrate\n",
      "dependent \\\\\n",
      "rrsG & 16S ribosomal RNA & translation & nan \\\\\n",
      "fliD & flagellum & cell adhesion, bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility & nan \\\\\n",
      "purR & DNA-binding transcriptional\n",
      "repressor PurR-hypoxanthine & regulation of DNA-templated\n",
      "transcription, purine\n",
      "nucleotide biosynthetic\n",
      "process, negative regulation\n",
      "of purine nucleotide\n",
      "biosynthetic process, negative\n",
      "regulation of DNA-templated\n",
      "transcription & nan \\\\\n",
      "appY & DLP12 prophage; DNA-binding\n",
      "transcriptional activator AppY & regulation of DNA-templated\n",
      "transcription, response to\n",
      "X-ray, cellular response to\n",
      "phosphate starvation, positive\n",
      "regulation of DNA-templated\n",
      "transcription & nan \\\\\n",
      "ftnB & putative ferritin-like protein & intracellular sequestering of\n",
      "iron ion & nan \\\\\n",
      "folX & dihydroneopterin triphosphate\n",
      "2'-epimerase & pteridine-containing compound\n",
      "biosynthetic process, folic\n",
      "acid-containing compound\n",
      "metabolic process & tetrahydromonapterin\n",
      "biosynthesis \\\\\n",
      "argH & argininosuccinate lyase & arginine biosynthetic process\n",
      "via ornithine, arginine\n",
      "biosynthetic process & superpathway of arginine and\n",
      "polyamine biosynthesis,\n",
      "L-arginine biosynthesis I (via\n",
      "L-ornithine) \\\\\n",
      "nrdA & ribonucleoside-diphosphate\n",
      "reductase 1 & protein folding,\n",
      "2'-deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "ribonucleoside diphosphate\n",
      "metabolic process,\n",
      "deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "nucleobase-containing small\n",
      "molecule interconversion & superpathway of pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis (<i>E.\n",
      "coli</i>), superpathway of\n",
      "pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis,\n",
      "pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis I,\n",
      "superpathway of adenosine\n",
      "nucleotides <i>de novo</i>\n",
      "biosynthesis II, adenosine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis II,\n",
      "superpathway of histidine,\n",
      "purine, and pyrimidine\n",
      "biosynthesis, superpathway of\n",
      "purine nucleotides <i>de\n",
      "novo</i> biosynthesis II,\n",
      "superpathway of guanosine\n",
      "nucleotides <i>de novo</i>\n",
      "biosynthesis II, guanosine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis II \\\\\n",
      "mdtF & multidrug efflux pump MdtEF-\n",
      "TolC & xenobiotic transmembrane\n",
      "transport, transmembrane\n",
      "transport, xenobiotic\n",
      "transport, response to toxic\n",
      "substance, bile acid and bile\n",
      "salt transport, response to\n",
      "antibiotic & nan \\\\\n",
      "gpr & L-glyceraldehyde 3-phosphate\n",
      "reductase & DNA damage response,\n",
      "methylglyoxal catabolic\n",
      "process & superpathway of methylglyoxal\n",
      "degradation, methylglyoxal\n",
      "degradation III \\\\\n",
      "ycjG & L-Ala-D/L-Glu epimerase & cell wall organization, amino\n",
      "acid catabolic process,\n",
      "peptidoglycan turnover & muropeptide degradation \\\\\n",
      "deaD & ATP-dependent RNA helicase\n",
      "DeaD & response to cold, mRNA\n",
      "stabilization, response to\n",
      "radiation, positive regulation\n",
      "of translation, ribosomal\n",
      "large subunit assembly, RNA\n",
      "catabolic process, cellular\n",
      "response to cold & nan \\\\\n",
      "appC & cytochrome <i>bd</i>-II\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase & transmembrane transport,\n",
      "aerobic electron transport\n",
      "chain, oxidative\n",
      "phosphorylation & nan \\\\\n",
      "cyoA & cytochrome\n",
      "<i>bo<sub>3</sub></i>\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase & ATP synthesis coupled electron\n",
      "transport, electron transport\n",
      "chain, aerobic respiration,\n",
      "electron transport coupled\n",
      "proton transport, aerobic\n",
      "electron transport chain & proline to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer, NADH to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer II, NADH to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer I,\n",
      "glycerol-3-phosphate to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer, D-lactate\n",
      "to cytochrome <i>bo</i>\n",
      "oxidase electron transfer,\n",
      "pyruvate to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer, succinate to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer \\\\\n",
      "metA & homoserine\n",
      "O-succinyltransferase & L-methionine biosynthetic\n",
      "process from homoserine via\n",
      "O-succinyl-L-homoserine and\n",
      "cystathionine, methionine\n",
      "biosynthetic process & aspartate superpathway,\n",
      "superpathway of L-lysine,\n",
      "L-threonine and L-methionine\n",
      "biosynthesis I, superpathway\n",
      "of L-homoserine and\n",
      "L-methionine biosynthesis,\n",
      "superpathway of\n",
      "<i>S</i>-adenosyl-L-methionine\n",
      "biosynthesis, L-methionine\n",
      "biosynthesis I \\\\\n",
      "ymdF & stress-induced bacterial\n",
      "acidophilic repeat motifs-\n",
      "containing protein YmdF & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation & nan \\\\\n",
      "lolC & lipoprotein release complex & protein localization to outer\n",
      "membrane, lipoprotein\n",
      "transport, lipoprotein\n",
      "localization to outer membrane & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria) \\\\\n",
      "rho & transcription termination\n",
      "factor Rho & DNA-templated transcription\n",
      "termination & nan \\\\\n",
      "ssrA & tmRNA & rescue of stalled ribosome,\n",
      "trans-translation-dependent\n",
      "protein tagging, trans-\n",
      "translation & nan \\\\\n",
      "yibJ & putative RHS domain-containing\n",
      "protein YibJ & nan & nan \\\\\n",
      "intA & CP4-57 prophage; integrase\n",
      "IntA & viral genome integration into\n",
      "host DNA, symbiont entry into\n",
      "host cell, establishment of\n",
      "integrated proviral latency,\n",
      "DNA integration, DNA\n",
      "recombination, provirus\n",
      "excision & nan \\\\\n",
      "cycA & D-\n",
      "serine/alanine/glycine:H<sup>+\n",
      "</sup>symporter & transmembrane transport, amino\n",
      "acid transport, D-alanine\n",
      "transmembrane transport,\n",
      "L-alanine transport, beta-\n",
      "alanine transport, glycine\n",
      "transport, D-serine\n",
      "transmembrane transport & nan \\\\\n",
      "abrB & putative regulator & regulation of gene expression & nan \\\\\n",
      "zupT & divalent metal ion transporter\n",
      "ZupT & iron ion transport, zinc ion\n",
      "transport, cobalt ion\n",
      "transport, zinc ion import\n",
      "across plasma membrane, iron\n",
      "ion import across plasma\n",
      "membrane, zinc ion\n",
      "transmembrane transport,\n",
      "transmembrane transport, metal\n",
      "ion transport & nan \\\\\n",
      "ansP & L-asparagine transporter & transmembrane transport, amino\n",
      "acid transport & nan \\\\\n",
      "wbbK & putative glycosyltransferase\n",
      "WbbK & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "wzzB & O-antigen co-polymerase & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "murD & UDP-<i>N</i>-acetylmuramoyl-L-\n",
      "alanine\\&mdash;D-glutamate\n",
      "ligase & cell wall organization,\n",
      "biosynthetic process, cell\n",
      "division, regulation of cell\n",
      "shape, peptidoglycan\n",
      "biosynthetic process & peptidoglycan biosynthesis I\n",
      "(<I>meso</I>-diaminopimelate\n",
      "containing),\n",
      "UDP-<i>N</i>-acetylmuramoyl-\n",
      "pentapeptide biosynthesis I\n",
      "(<i>meso</i>-diaminopimelate\n",
      "containing) \\\\\n",
      "rodZ & cytoskeleton protein RodZ & peptidoglycan biosynthetic\n",
      "process, FtsZ-dependent\n",
      "cytokinesis, regulation of\n",
      "cell shape & nan \\\\\n",
      "mutL & methyl-directed mismatch\n",
      "repair system & nucleotide-excision repair,\n",
      "DNA duplex unwinding,\n",
      "regulation of DNA\n",
      "recombination, DNA repair,\n",
      "mismatch repair involved in\n",
      "maintenance of fidelity\n",
      "involved in DNA-dependent DNA\n",
      "replication, mismatch repair & nan \\\\\n",
      "yibX & protein YibX-S & nan & nan \\\\\n",
      "def & peptide deformylase & translation, co-translational\n",
      "protein modification,\n",
      "N-terminal protein amino acid\n",
      "modification & nan \\\\\n",
      "lspA & lipoprotein signal peptidase & proteolysis, lipoprotein\n",
      "biosynthetic process via\n",
      "signal peptide cleavage & lipoprotein posttranslational\n",
      "modification (Gram-negative\n",
      "bacteria) \\\\\n",
      "cbrC & PF03691 family colicin E2\n",
      "tolerance protein CbrC & bacteriocin immunity & nan \\\\\n",
      "sgcX & KpLE2 phage-like element;\n",
      "putative endoglucanase with\n",
      "Zn-dependent exopeptidase\n",
      "domain & proteolysis & nan \\\\\n",
      "abpA & CP4-57 prophage; anti-\n",
      "bacteriophage protein & defense response to virus & nan \\\\\n",
      "intZ & CPZ-55 prophage; putative\n",
      "phage integrase IntZ & viral genome integration into\n",
      "host DNA, establishment of\n",
      "integrated proviral latency,\n",
      "symbiont entry into host cell,\n",
      "DNA integration, DNA\n",
      "recombination & nan \\\\\n",
      "rbsB & ribose ABC transporter & chemotaxis, D-ribose\n",
      "transmembrane transport,\n",
      "positive chemotaxis & nan \\\\\n",
      "rpe & ribulose-phosphate 3-epimerase & pentose-phosphate shunt,\n",
      "carbohydrate metabolic\n",
      "process, pentose catabolic\n",
      "process, pentose-phosphate\n",
      "shunt, non-oxidative branch & pentose phosphate pathway,\n",
      "pentose phosphate pathway\n",
      "(non-oxidative branch) I \\\\\n",
      "tar & chemotaxis signaling complex\n",
      "core unit - aspartate sensing & positive regulation of post-\n",
      "translational protein\n",
      "modification, cellular\n",
      "response to amino acid\n",
      "stimulus, regulation of\n",
      "chemotaxis, detection of\n",
      "chemical stimulus, signal\n",
      "complex assembly, signal\n",
      "transduction, protein\n",
      "homooligomerization,\n",
      "chemotaxis & nan \\\\\n",
      "rnlA & CP4-57 prophage; RNase LS,\n",
      "toxin of the RnlAB toxin-\n",
      "antitoxin system, RnlA-RnlB\n",
      "toxin-antitoxin complex & regulation of DNA-templated\n",
      "transcription, single-species\n",
      "biofilm formation, regulation\n",
      "of growth, defense response to\n",
      "virus, mRNA catabolic process & nan \\\\\n",
      "hyfF & hydrogenase 4 & anaerobic electron transport\n",
      "chain, formate oxidation,\n",
      "anaerobic respiration, glucose\n",
      "catabolic process, DNA damage\n",
      "response & nan \\\\\n",
      "rcsA & RcsAB DNA-binding\n",
      "transcriptional dual regulator & positive regulation of DNA-\n",
      "templated transcription,\n",
      "negative regulation of DNA-\n",
      "templated transcription,\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility, regulation of\n",
      "capsule organization,\n",
      "regulation of DNA-templated\n",
      "transcription, DNA-templated\n",
      "transcription & nan \\\\\n",
      "ychO & putative invasin YchO & nan & nan \\\\\n",
      "pgpB & phosphatidylglycerophosphatase\n",
      "B & phospholipid catabolic\n",
      "process, phosphatidylglycerol\n",
      "biosynthetic process,\n",
      "peptidoglycan biosynthetic\n",
      "process, glycerophospholipid\n",
      "biosynthetic process & superpathway of phospholipid\n",
      "biosynthesis III (<i>E.\n",
      "coli</i>), cardiolipin\n",
      "biosynthesis III, cardiolipin\n",
      "biosynthesis I, polyisoprenoid\n",
      "biosynthesis (<i>E. coli</i>),\n",
      "di-<i>trans</i>,poly-\n",
      "<i>cis</i>-undecaprenyl\n",
      "phosphate biosynthesis \\\\\n",
      "gyrA & DNA gyrase & chromosome organization,\n",
      "response to xenobiotic\n",
      "stimulus, negative regulation\n",
      "of DNA-templated DNA\n",
      "replication, DNA-templated DNA\n",
      "replication, DNA topological\n",
      "change, response to\n",
      "antibiotic, DNA-templated\n",
      "transcription & nan \\\\\n",
      "alaA & glutamate\\&mdash;pyruvate\n",
      "aminotransferase AlaA & amino acid biosynthetic\n",
      "process, biosynthetic process,\n",
      "alanine biosynthetic process,\n",
      "D-alanine biosynthetic\n",
      "process, DNA damage response,\n",
      "response to antibiotic,\n",
      "L-alanine biosynthetic process\n",
      "from pyruvate & superpathway of L-alanine\n",
      "biosynthesis, L-alanine\n",
      "biosynthesis II \\\\\n",
      "yciQ & DUF2207 domain-containing\n",
      "protein YciQ & nan & nan \\\\\n",
      "ppdC & prepilin-type N-terminal\n",
      "cleavage/methylation domain-\n",
      "containing protein PpdC & nan & nan \\\\\n",
      "ynaE & Rac prophage; uncharacterized\n",
      "protein YnaE & response to cold & nan \\\\\n",
      "oppA & oligopeptide ABC transporter & oligopeptide import across\n",
      "plasma membrane, transmembrane\n",
      "transport, peptide transport,\n",
      "protein transport, chaperone-\n",
      "mediated protein folding,\n",
      "response to heat, oligopeptide\n",
      "transport & nan \\\\\n",
      "acnB & bifunctional aconitate\n",
      "hydratase B/2-methylisocitrate\n",
      "dehydratase & glyoxylate cycle,\n",
      "tricarboxylic acid cycle,\n",
      "regulation of translation,\n",
      "propionate catabolic process,\n",
      "2-methylcitrate cycle & superpathway of glycolysis,\n",
      "pyruvate dehydrogenase, TCA,\n",
      "and glyoxylate bypass,\n",
      "superpathway of glyoxylate\n",
      "bypass and TCA, glyoxylate\n",
      "cycle, TCA cycle I\n",
      "(prokaryotic), mixed acid\n",
      "fermentation, 2-methylcitrate\n",
      "cycle I \\\\\n",
      "gltT & tRNA-Glu(UUC) & nan & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging \\\\\n",
      "ffh & Signal Recognition Particle & SRP-dependent cotranslational\n",
      "protein targeting to membrane,\n",
      "protein targeting to membrane & nan \\\\\n",
      "mntH & Mn(2+):H(+) symporter & iron ion transmembrane\n",
      "transport, proton\n",
      "transmembrane transport,\n",
      "cadmium ion transmembrane\n",
      "transport, manganese ion\n",
      "transport, iron ion transport,\n",
      "cobalt ion transport, cadmium\n",
      "ion transport, monoatomic ion\n",
      "transport, metal ion\n",
      "transport, manganese import\n",
      "into cell, cellular response\n",
      "to oxidative stress & nan \\\\\n",
      "ydhQ & putative adhesin-related\n",
      "protein YdhQ & DNA damage response & nan \\\\\n",
      "fliC & flagellum & nan & nan \\\\\n",
      "rrlE & 23S ribosomal RNA & translation & nan \\\\\n",
      "nadK & NAD kinase & NAD metabolic process, NADP\n",
      "biosynthetic process & NAD phosphorylation and\n",
      "dephosphorylation \\\\\n",
      "bax & putative glycoside hydrolase\n",
      "Bax & nan & nan \\\\\n",
      "rfbC & dTDP-4-dehydrorhamnose\n",
      "3,5-epimerase & dTDP-rhamnose biosynthetic\n",
      "process, extracellular\n",
      "polysaccharide biosynthetic\n",
      "process, lipopolysaccharide\n",
      "biosynthetic process, O\n",
      "antigen biosynthetic process,\n",
      "response to UV, response to\n",
      "antibiotic, DNA damage\n",
      "response & <i>O</i>-antigen building\n",
      "blocks biosynthesis (<i>E.\n",
      "coli</i>), dTDP-L-rhamnose\n",
      "biosynthesis \\\\\n",
      "radA & DNA recombination protein & response to radiation, DNA\n",
      "repair, recombinational repair & nan \\\\\n",
      "glf & UDP-galactopyranose mutase & lipopolysaccharide\n",
      "biosynthetic process & <i>O</i>-antigen building\n",
      "blocks biosynthesis (<i>E.\n",
      "coli</i>) \\\\\n",
      "trpD & anthranilate synthase subunit\n",
      "TrpE, anthranilate synthase\n",
      "subunit TrpD & phenazine biosynthetic\n",
      "process, glutamine metabolic\n",
      "process, tryptophan\n",
      "biosynthetic process & L-glutamine degradation I,\n",
      "L-glutamate biosynthesis I,\n",
      "ammonia assimilation cycle\n",
      "III, superpathway of\n",
      "chorismate metabolism,\n",
      "superpathway of aromatic amino\n",
      "acid biosynthesis,\n",
      "superpathway of L-tryptophan\n",
      "biosynthesis, L-tryptophan\n",
      "biosynthesis \\\\\n",
      "rrsB & 16S ribosomal RNA & translation & nan \\\\\n",
      "rtcR & DNA-binding transcriptional\n",
      "activator RtcR & regulation of DNA-templated\n",
      "transcription, phosphorelay\n",
      "signal transduction system,\n",
      "positive regulation of DNA-\n",
      "templated transcription & nan \\\\\n",
      "gabP & 4-aminobutanoate:H<sup>+</sup>\n",
      "symporter & transmembrane transport,\n",
      "gamma-aminobutyric acid\n",
      "catabolic process, DNA damage\n",
      "response, amino acid\n",
      "transport, cellular response\n",
      "to nitrogen starvation,\n",
      "response to carbon starvation,\n",
      "gamma-aminobutyric acid\n",
      "transport & nan \\\\\n",
      "entF & seryl-[EntF peptidyl-carrier\n",
      "protein] & secondary metabolite\n",
      "biosynthetic process,\n",
      "biosynthetic process, amino\n",
      "acid activation for\n",
      "nonribosomal peptide\n",
      "biosynthetic process,\n",
      "enterobactin biosynthetic\n",
      "process & superpathway of chorismate\n",
      "metabolism, acyl carrier\n",
      "protein metabolism,\n",
      "enterobactin biosynthesis \\\\\n",
      "yqiI & putative fimbrial protein YqiI & cell adhesion, response to\n",
      "methylglyoxal & nan \\\\\n",
      "idi & isopentenyl-diphosphate\n",
      "\\&Delta;-isomerase & dimethylallyl diphosphate\n",
      "biosynthetic process, DNA\n",
      "damage response, isoprenoid\n",
      "biosynthetic process & methylerythritol phosphate\n",
      "pathway I \\\\\n",
      "gltW & tRNA-Glu(UUC) & nan & polycistronic tRNA processing\n",
      "I, monocistronic tRNA\n",
      "processing I, monocistronic\n",
      "tRNA processing II,\n",
      "monocistronic tRNA processing\n",
      "III, tRNA processing,\n",
      "uroporphyrinogen-III I (from\n",
      "glutamate), tRNA charging \\\\\n",
      "rrlG & 23S ribosomal RNA & translation & nan \\\\\n",
      "dgcQ & diguanylate cyclase DgcQ & cell adhesion involved in\n",
      "single-species biofilm\n",
      "formation, cellulose\n",
      "biosynthetic process, negative\n",
      "regulation of bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility & nan \\\\\n",
      "rrsA & ribosome & nan & nan \\\\\n",
      "ynjF & CDP-alcohol\n",
      "phosphatidyltransferase\n",
      "domain-containing protein YnjF & phospholipid biosynthetic\n",
      "process & nan \\\\\n",
      "aldA & aldehyde dehydrogenase A, NAD-\n",
      "linked & gamma-aminobutyric acid\n",
      "catabolic process, fucose\n",
      "catabolic process, rhamnose\n",
      "catabolic process, L-fucose\n",
      "catabolic process & superpathway of fucose and\n",
      "rhamnose degradation,\n",
      "methylglyoxal degradation IV,\n",
      "superpathway of methylglyoxal\n",
      "degradation, L-lactaldehyde\n",
      "degradation (aerobic),\n",
      "superpathway of glycol\n",
      "metabolism and degradation,\n",
      "ethylene glycol degradation,\n",
      "D-arabinose degradation II \\\\\n",
      "rrlD & 23S ribosomal RNA & translation & nan \\\\\n",
      "dnaK & chaperone protein DnaK & protein refolding, protein-\n",
      "containing complex assembly,\n",
      "negative regulation of DNA-\n",
      "templated transcription, DNA\n",
      "replication, stress response\n",
      "to copper ion, protein\n",
      "folding, chaperone cofactor-\n",
      "dependent protein refolding,\n",
      "cellular response to unfolded\n",
      "protein, protein unfolding,\n",
      "response to heat & nan \\\\\n",
      "yohC & putative inner membrane\n",
      "protein & DNA damage response & nan \\\\\n",
      "nanY & KpLE2 phage-like element; 2,7-\n",
      "anhydro-<i>N</i>-\n",
      "acetylneuraminate hydratase & DNA damage response,\n",
      "N-acetylneuraminate catabolic\n",
      "process & nan \\\\\n",
      "ynfD & DUF1161 domain-containing\n",
      "protein YnfD & nan & nan \\\\\n",
      "wbbH & putative O-antigen polymerase & lipopolysaccharide\n",
      "biosynthetic process & nan \\\\\n",
      "puuB & \\&gamma;-glutamylputrescine\n",
      "oxidase & putrescine catabolic process & superpathway of L-arginine and\n",
      "L-ornithine degradation,\n",
      "superpathway of ornithine\n",
      "degradation, superpathway of\n",
      "L-arginine, putrescine, and\n",
      "4-aminobutanoate degradation,\n",
      "putrescine degradation II \\\\\n",
      "fhuA & ferrichrome outer membrane\n",
      "transport complex & iron ion transport,\n",
      "intracellular iron ion\n",
      "homeostasis, siderophore-\n",
      "dependent iron import into\n",
      "cell, siderophore transport,\n",
      "siderophore transmembrane\n",
      "transport & nan \\\\\n",
      "zapA & cell division protein ZapA & cell septum assembly, septin\n",
      "ring assembly, division septum\n",
      "assembly, FtsZ-dependent\n",
      "cytokinesis, cell division & nan \\\\\n",
      "rsmJ & 16S rRNA m<sup>2</sup>G1516\n",
      "methyltransferase & methylation, rRNA processing,\n",
      "rRNA methylation, rRNA base\n",
      "methylation & nan \\\\\n",
      "efeU & inactive ferrous iron permease\n",
      "EfeU & iron ion transmembrane\n",
      "transport & nan \\\\\n",
      "narQ & sensor histidine kinase NarQ & phosphorelay signal\n",
      "transduction system, signal\n",
      "transduction, nitrate\n",
      "assimilation, cellular\n",
      "response to nitrate, cellular\n",
      "response to nitrite & NarQ Two-Component Signal\n",
      "Transduction System, nitrate\n",
      "dependent \\\\\n",
      "rrsG & 16S ribosomal RNA & translation & nan \\\\\n",
      "fliD & flagellum & cell adhesion, bacterial-type\n",
      "flagellum-dependent cell\n",
      "motility & nan \\\\\n",
      "purR & DNA-binding transcriptional\n",
      "repressor PurR-hypoxanthine & regulation of DNA-templated\n",
      "transcription, purine\n",
      "nucleotide biosynthetic\n",
      "process, negative regulation\n",
      "of purine nucleotide\n",
      "biosynthetic process, negative\n",
      "regulation of DNA-templated\n",
      "transcription & nan \\\\\n",
      "appY & DLP12 prophage; DNA-binding\n",
      "transcriptional activator AppY & regulation of DNA-templated\n",
      "transcription, response to\n",
      "X-ray, cellular response to\n",
      "phosphate starvation, positive\n",
      "regulation of DNA-templated\n",
      "transcription & nan \\\\\n",
      "ftnB & putative ferritin-like protein & intracellular sequestering of\n",
      "iron ion & nan \\\\\n",
      "folX & dihydroneopterin triphosphate\n",
      "2'-epimerase & pteridine-containing compound\n",
      "biosynthetic process, folic\n",
      "acid-containing compound\n",
      "metabolic process & tetrahydromonapterin\n",
      "biosynthesis \\\\\n",
      "argH & argininosuccinate lyase & arginine biosynthetic process\n",
      "via ornithine, arginine\n",
      "biosynthetic process & superpathway of arginine and\n",
      "polyamine biosynthesis,\n",
      "L-arginine biosynthesis I (via\n",
      "L-ornithine) \\\\\n",
      "nrdA & ribonucleoside-diphosphate\n",
      "reductase 1 & protein folding,\n",
      "2'-deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "ribonucleoside diphosphate\n",
      "metabolic process,\n",
      "deoxyribonucleotide\n",
      "biosynthetic process,\n",
      "nucleobase-containing small\n",
      "molecule interconversion & superpathway of pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis (<i>E.\n",
      "coli</i>), superpathway of\n",
      "pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis,\n",
      "pyrimidine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis I,\n",
      "superpathway of adenosine\n",
      "nucleotides <i>de novo</i>\n",
      "biosynthesis II, adenosine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis II,\n",
      "superpathway of histidine,\n",
      "purine, and pyrimidine\n",
      "biosynthesis, superpathway of\n",
      "purine nucleotides <i>de\n",
      "novo</i> biosynthesis II,\n",
      "superpathway of guanosine\n",
      "nucleotides <i>de novo</i>\n",
      "biosynthesis II, guanosine\n",
      "deoxyribonucleotides <i>de\n",
      "novo</i> biosynthesis II \\\\\n",
      "mdtF & multidrug efflux pump MdtEF-\n",
      "TolC & xenobiotic transmembrane\n",
      "transport, transmembrane\n",
      "transport, xenobiotic\n",
      "transport, response to toxic\n",
      "substance, bile acid and bile\n",
      "salt transport, response to\n",
      "antibiotic & nan \\\\\n",
      "gpr & L-glyceraldehyde 3-phosphate\n",
      "reductase & DNA damage response,\n",
      "methylglyoxal catabolic\n",
      "process & superpathway of methylglyoxal\n",
      "degradation, methylglyoxal\n",
      "degradation III \\\\\n",
      "ycjG & L-Ala-D/L-Glu epimerase & cell wall organization, amino\n",
      "acid catabolic process,\n",
      "peptidoglycan turnover & muropeptide degradation \\\\\n",
      "deaD & ATP-dependent RNA helicase\n",
      "DeaD & response to cold, mRNA\n",
      "stabilization, response to\n",
      "radiation, positive regulation\n",
      "of translation, ribosomal\n",
      "large subunit assembly, RNA\n",
      "catabolic process, cellular\n",
      "response to cold & nan \\\\\n",
      "appC & cytochrome <i>bd</i>-II\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase & transmembrane transport,\n",
      "aerobic electron transport\n",
      "chain, oxidative\n",
      "phosphorylation & nan \\\\\n",
      "cyoA & cytochrome\n",
      "<i>bo<sub>3</sub></i>\n",
      "ubiquinol:oxygen\n",
      "oxidoreductase & ATP synthesis coupled electron\n",
      "transport, electron transport\n",
      "chain, aerobic respiration,\n",
      "electron transport coupled\n",
      "proton transport, aerobic\n",
      "electron transport chain & proline to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer, NADH to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer II, NADH to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer I,\n",
      "glycerol-3-phosphate to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer, D-lactate\n",
      "to cytochrome <i>bo</i>\n",
      "oxidase electron transfer,\n",
      "pyruvate to cytochrome\n",
      "<i>bo</i> oxidase electron\n",
      "transfer, succinate to\n",
      "cytochrome <i>bo</i> oxidase\n",
      "electron transfer \\\\\n",
      "metA & homoserine\n",
      "O-succinyltransferase & L-methionine biosynthetic\n",
      "process from homoserine via\n",
      "O-succinyl-L-homoserine and\n",
      "cystathionine, methionine\n",
      "biosynthetic process & aspartate superpathway,\n",
      "superpathway of L-lysine,\n",
      "L-threonine and L-methionine\n",
      "biosynthesis I, superpathway\n",
      "of L-homoserine and\n",
      "L-methionine biosynthesis,\n",
      "superpathway of\n",
      "<i>S</i>-adenosyl-L-methionine\n",
      "biosynthesis, L-methionine\n",
      "biosynthesis I \\\\\n",
      "\\bottomrule\n",
      "\\end{longtable}\n",
      "\n",
      "\\endgroup\n",
      "\n"
     ]
    }
   ],
   "source": [
    "Data\n",
    "\n",
    "table = GT(Data).cols_width({\"Genes\":\"20%\", \"Products\":\"20%\", \"Biological Process\":\"20%\", \"Pathways\":\"20%\"})\n",
    "print(table.as_latex(use_longtable=True, tbl_pos='!t'))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "dCasRx",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
